<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.18"/>
<meta name="viewport" content="width=device-width, initial-scale=1"/>
<title>AMICI: References</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="navtree.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="resize.js"></script>
<script type="text/javascript" src="navtreedata.js"></script>
<script type="text/javascript" src="navtree.js"></script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/searchdata.js"></script>
<script type="text/javascript" src="search/search.js"></script>
<link href="customdoxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td id="projectalign" style="padding-left: 0.5em;">
   <div id="projectname">AMICI
   </div>
   <div id="projectbrief">Advanced Multilanguage Interface for CVODES and IDAS</div>
  </td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.18 -->
<script type="text/javascript">
/* @license magnet:?xt=urn:btih:cf05388f2679ee054f2beb29a391d25f4e673ac3&amp;dn=gpl-2.0.txt GPL-v2 */
var searchBox = new SearchBox("searchBox", "search",false,'Search');
/* @license-end */
</script>
<script type="text/javascript" src="menudata.js"></script>
<script type="text/javascript" src="menu.js"></script>
<script type="text/javascript">
/* @license magnet:?xt=urn:btih:cf05388f2679ee054f2beb29a391d25f4e673ac3&amp;dn=gpl-2.0.txt GPL-v2 */
$(function() {
  initMenu('',true,false,'search.php','Search');
  $(document).ready(function() { init_search(); });
});
/* @license-end */</script>
<div id="main-nav"></div>
</div><!-- top -->
<div id="side-nav" class="ui-resizable side-nav-resizable">
  <div id="nav-tree">
    <div id="nav-tree-contents">
      <div id="nav-sync" class="sync"></div>
    </div>
  </div>
  <div id="splitbar" style="-moz-user-select:none;" 
       class="ui-resizable-handle">
  </div>
</div>
<script type="text/javascript">
/* @license magnet:?xt=urn:btih:cf05388f2679ee054f2beb29a391d25f4e673ac3&amp;dn=gpl-2.0.txt GPL-v2 */
$(document).ready(function(){initNavTree('md_documentation_references.html',''); initResizable(); });
/* @license-end */
</script>
<div id="doc-content">
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
</div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="PageDoc"><div class="header">
  <div class="headertitle">
<div class="title">References </div>  </div>
</div><!--header-->
<div class="contents">
<div class="textblock"><p>List of publications using AMICI. Total number is 35.</p>
<p>If you applied AMICI in your work and your publication is missing, please let us know via a new Github issue.</p>
<h1 id="section" class="unnumbered">2019</h1>
<div id="refs" class="references"> <div id="ref-DharmarajanKal2019"> </div></div><div id="refs" class="references"><div id="ref-DharmarajanKal2019">Dharmarajan, Lekshmi, Hans-Michael Kaltenbach, Fabian Rudolf, and Joerg Stelling. 2019. “A Simple and Flexible Computational Framework for Inferring Sources of Heterogeneity from Single-Cell Dynamics.” <em>Cell Systems</em> 8 (1). Elsevier: 15–26.e11. <a href="https://doi.org/10.1016/j.cels.2018.12.007" class="uri">https://doi.org/10.1016/j.cels.2018.12.007</a>.</div></div><div id="refs" class="references"><div id="ref-DharmarajanKal2019"> </div> <div id="ref-FischerFie2017"> </div></div><div id="refs" class="references"><div id="ref-FischerFie2017">Fischer, David S., Anna K. Fiedler, Eric Kernfeld, Ryan M. J. Genga, Aimée Bastidas-Ponce, Mostafa Bakhti, Heiko Lickert, Jan Hasenauer, Rene Maehr, and Fabian J. Theis. 2019. “Inferring Population Dynamics from Single-Cell Rna-Sequencing Time Series Data.” <em>Nature Biotechnology</em> 37: 461–68. <a href="https://doi.org/10.1038/s41587-019-0088-0" class="uri">https://doi.org/10.1038/s41587-019-0088-0</a>.</div></div><div id="refs" class="references"><div id="ref-FischerFie2017"> </div> <div id="ref-GreggSar2019"> </div></div><div id="refs" class="references"><div id="ref-GreggSar2019">Gregg, Robert W, Saumendra N Sarkar, and Jason E Shoemaker. 2019. “Mathematical Modeling of the cGAS Pathway Reveals Robustness of Dna Sensing to Trex1 Feedback.” <em>Journal of Theoretical Biology</em> 462 (February): 148–57. <a href="https://doi.org/10.1016/j.jtbi.2018.11.001" class="uri">https://doi.org/10.1016/j.jtbi.2018.11.001</a>.</div></div><div id="refs" class="references"><div id="ref-GreggSar2019"> </div> <div id="ref-KapferSta2019"> </div></div><div id="refs" class="references"><div id="ref-KapferSta2019">Kapfer, Eva-Maria, Paul Stapor, and Jan Hasenauer. 2019. “Challenges in the Calibration of Large-Scale Ordinary Differential Equation Models.” <em>Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (FOSBE)</em>.</div></div><div id="refs" class="references"><div id="ref-KapferSta2019"> </div> <div id="ref-LinesPas2019"> </div></div><div id="refs" class="references"><div id="ref-LinesPas2019">Lines, Glenn Terje, Lukasz Paszkowski, Leonard Schmiester, Daniel Weindl, Paul Stapor, and Jan Hasenauer. 2019. “Efficient Computation of Steady States in Large-Scale Ode Models of Biochemical Reaction Networks.” <em>Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (FOSBE)</em>.</div></div><div id="refs" class="references"><div id="ref-LinesPas2019"> </div> <div id="ref-NousiainenInt2019"> </div></div><div id="refs" class="references"><div id="ref-NousiainenInt2019">Nousiainen, Kari, Jukka Intosalmi, and Harri Lähdesmäki. 2019. “A Mathematical Model for Enhancer Activation Kinetics During Cell Differentiation.” In <em>Algorithms for Computational Biology</em>, edited by Ian Holmes, Carlos Martín-Vide, and Miguel A. Vega-Rodríguez, 191–202. Cham: Springer International Publishing.</div></div><div id="refs" class="references"><div id="ref-NousiainenInt2019"> </div> <div id="ref-PedretscherKal2019"> </div></div><div id="refs" class="references"><div id="ref-PedretscherKal2019">Pedretscher, B., B. Kaltenbacher, and O. Pfeiler. 2019. “Parameter Identification and Uncertainty Quantification in Stochastic State Space Models and Its Application to Texture Analysis.” <em>Applied Numerical Mathematics</em> 146: 38–54. <a href="https://doi.org/https://doi.org/10.1016/j.apnum.2019.06.020" class="uri">https://doi.org/https://doi.org/10.1016/j.apnum.2019.06.020</a>.</div></div><div id="refs" class="references"><div id="ref-PedretscherKal2019"> </div> <div id="ref-PittBan2019"> </div></div><div id="refs" class="references"><div id="ref-PittBan2019">Pitt, Jake Alan, and Julio R Banga. 2019. “Parameter Estimation in Models of Biological Oscillators: An Automated Regularised Estimation Approach.” <em>BMC Bioinformatics</em> 20 (1): 82. <a href="https://doi.org/10.1186/s12859-019-2630-y" class="uri">https://doi.org/10.1186/s12859-019-2630-y</a>.</div></div><div id="refs" class="references"><div id="ref-PittBan2019"> </div> <div id="ref-SchmiesterSch2019"> </div></div><div id="refs" class="references"><div id="ref-SchmiesterSch2019">Schmiester, Leonard, Yannik Schälte, Fabian Fröhlich, Jan Hasenauer, and Daniel Weindl. 2019. “Efficient parameterization of large-scale dynamic models based on relative measurements.” <em>Bioinformatics</em>, July. <a href="https://doi.org/10.1093/bioinformatics/btz581" class="uri">https://doi.org/10.1093/bioinformatics/btz581</a>.</div></div><div id="refs" class="references"><div id="ref-SchmiesterSch2019"> </div> <div id="ref-SchmiesterWei2019"> </div></div><div id="refs" class="references"><div id="ref-SchmiesterWei2019">Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2019. “Statistical Inference of Mechanistic Models from Qualitative Data Using an Efficient Optimal Scaling Approach.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory. <a href="https://doi.org/10.1101/848648" class="uri">https://doi.org/10.1101/848648</a>.</div></div><div id="refs" class="references"><div id="ref-SchmiesterWei2019"> </div> <div id="ref-StaporSch2019"> </div></div><div id="refs" class="references"><div id="ref-StaporSch2019">Stapor, Paul, Leonard Schmiester, Christoph Wierling, Bodo Lange, Daniel Weindl, and Jan Hasenauer. 2019. “Mini-Batch Optimization Enables Training of Ode Models on Large-Scale Datasets.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory. <a href="https://doi.org/10.1101/859884" class="uri">https://doi.org/10.1101/859884</a>.</div></div><div id="refs" class="references"><div id="ref-StaporSch2019"> </div> <div id="ref-VillaverdeRai2019"> </div></div><div id="refs" class="references"><div id="ref-VillaverdeRai2019">Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R. Banga. 2019. “A Comparison of Methods for Quantifying Prediction Uncertainty in Systems Biology.” <em>Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (FOSBE)</em>.</div></div><div id="refs" class="references"><div id="ref-VillaverdeRai2019"> </div> <div id="ref-WangSta2019"> </div></div><div id="refs" class="references"><div id="ref-WangSta2019">Wang, Dantong, Paul Stapor, and Jan Hasenauer. 2019. “Dirac Mixture Distributions for the Approximation of Mixed Effects Models.” <em>Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (FOSBE)</em>.</div></div><div id="refs" class="references"><div id="ref-WangSta2019"> </div> </div> <h1 id="section" class="unnumbered">2018</h1>
<div id="refs" class="references"> <div id="ref-BallnusSch2018"> </div></div><div id="refs" class="references"><div id="ref-BallnusSch2018">Ballnus, Benjamin, Steffen Schaper, Fabian J Theis, and Jan Hasenauer. 2018. “Bayesian Parameter Estimation for Biochemical Reaction Networks Using Region-Based Adaptive Parallel Tempering.” <em>Bioinformatics</em> 34 (13): i494–i501. <a href="https://doi.org/10.1093/bioinformatics/bty229" class="uri">https://doi.org/10.1093/bioinformatics/bty229</a>.</div></div><div id="refs" class="references"><div id="ref-BallnusSch2018"> </div> <div id="ref-BastCal2018"> </div></div><div id="refs" class="references"><div id="ref-BastCal2018">Bast, Lisa, Filippo Calzolari, Michael Strasser, Jan Hasenauer, Fabian J. Theis, Jovica Ninkovic, and Carsten Marr. 2018. “Subtle Changes in Clonal Dynamics Underlie the Age-Related Decline in Neurogenesis.” <em>Cell Reports</em>.</div></div><div id="refs" class="references"><div id="ref-BastCal2018"> </div> <div id="ref-FroehlichKes2018"> </div></div><div id="refs" class="references"><div id="ref-FroehlichKes2018">Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6). Elsevier: 567–579.e6. <a href="https://doi.org/10.1016/j.cels.2018.10.013" class="uri">https://doi.org/10.1016/j.cels.2018.10.013</a>.</div></div><div id="refs" class="references"><div id="ref-FroehlichKes2018"> </div> <div id="ref-FroehlichRei2018"> </div></div><div id="refs" class="references"><div id="ref-FroehlichRei2018">Fröhlich, Fabian, Anita Reiser, Laura Fink, Daniel Woschée, Thomas Ligon, Fabian Joachim Theis, Joachim Oskar Rädler, and Jan Hasenauer. 2018. “Multi-Experiment Nonlinear Mixed Effect Modeling of Single-Cell Translation Kinetics After Transfection.” <em>Npj Systems Biology and Applications</em> 5 (1): 1. <a href="https://doi.org/10.1038/s41540-018-0079-7" class="uri">https://doi.org/10.1038/s41540-018-0079-7</a>.</div></div><div id="refs" class="references"><div id="ref-FroehlichRei2018"> </div> <div id="ref-KaltenbacherPed2018"> </div></div><div id="refs" class="references"><div id="ref-KaltenbacherPed2018">Kaltenbacher, Barbara, and Barbara Pedretscher. 2018. “Parameter Estimation in Sdes via the Fokker–Planck Equation: Likelihood Function and Adjoint Based Gradient Computation.” <em>Journal of Mathematical Analysis and Applications</em> 465 (2): 872–84. <a href="https://doi.org/https://doi.org/10.1016/j.jmaa.2018.05.048" class="uri">https://doi.org/https://doi.org/10.1016/j.jmaa.2018.05.048</a>.</div></div><div id="refs" class="references"><div id="ref-KaltenbacherPed2018"> </div> <div id="ref-LoosKra2018"> </div></div><div id="refs" class="references"><div id="ref-LoosKra2018">Loos, Carolin, Sabrina Krause, and Jan Hasenauer. 2018. “Hierarchical Optimization for the Efficient Parametrization of ODE Models.” <em>Bioinformatics</em> 34 (24): 4266–73. <a href="https://doi.org/10.1093/bioinformatics/bty514" class="uri">https://doi.org/10.1093/bioinformatics/bty514</a>.</div></div><div id="refs" class="references"><div id="ref-LoosKra2018"> </div> <div id="ref-LoosMoe2018"> </div></div><div id="refs" class="references"><div id="ref-LoosMoe2018">Loos, Carolin, Katharina Moeller, Fabian Fröhlich, Tim Hucho, and Jan Hasenauer. 2018. “A Hierarchical, Data-Driven Approach to Modeling Single-Cell Populations Predicts Latent Causes of Cell-to-Cell Variability.” <em>Cell Systems</em> 6 (5). Elsevier: 593–603. <a href="https://doi.org/10.1016/j.cels.2018.04.008" class="uri">https://doi.org/10.1016/j.cels.2018.04.008</a>.</div></div><div id="refs" class="references"><div id="ref-LoosMoe2018"> </div> <div id="ref-SchaelteSta2018"> </div></div><div id="refs" class="references"><div id="ref-SchaelteSta2018">Schälte, Y., P. Stapor, and J. Hasenauer. 2018. “Evaluation of Derivative-Free Optimizers for Parameter Estimation in Systems Biology.” <em>FAC-PapersOnLine</em> 51 (19): 98–101.</div></div><div id="refs" class="references"><div id="ref-SchaelteSta2018"> </div> <div id="ref-StaporFro2018"> </div></div><div id="refs" class="references"><div id="ref-StaporFro2018">Stapor, Paul, Fabian Fröhlich, and Jan Hasenauer. 2018. “Optimization and Profile Calculation of ODE Models Using Second Order Adjoint Sensitivity Analysis.” <em>Bioinformatics</em> 34 (13). Oxford University Press: i151–i159.</div></div><div id="refs" class="references"><div id="ref-StaporFro2018"> </div> <div id="ref-VillaverdeFro2018"> </div></div><div id="refs" class="references"><div id="ref-VillaverdeFro2018">Villaverde, Alejandro F., Fabian Froehlich, Daniel Weindl, Jan Hasenauer, and Julio R Banga. 2018. “Benchmarking Optimization Methods for Parameter Estimation in Large Kinetic Models.” <em>Bioinformatics</em>, bty736.</div></div><div id="refs" class="references"><div id="ref-VillaverdeFro2018"> </div> </div> <h1 id="section" class="unnumbered">2017</h1>
<div id="refs" class="references"> <div id="ref-BallnusHug2017"> </div></div><div id="refs" class="references"><div id="ref-BallnusHug2017">Ballnus, B., S. Hug, K. Hatz, L. Görlitz, J. Hasenauer, and F. J. Theis. 2017. “Comprehensive Benchmarking of Markov Chain Monte Carlo Methods for Dynamical Systems.” <em>BMC Syst. Biol.</em> 11 (63). <a href="https://doi.org/10.1186/s12918-017-0433-1" class="uri">https://doi.org/10.1186/s12918-017-0433-1</a>.</div></div><div id="refs" class="references"><div id="ref-BallnusHug2017"> </div> <div id="ref-FroehlichKal2017"> </div></div><div id="refs" class="references"><div id="ref-FroehlichKal2017">Fröhlich, F., B. Kaltenbacher, F. J. Theis, and J. Hasenauer. 2017. “Scalable Parameter Estimation for Genome-Scale Biochemical Reaction Networks.” <em>PLoS Comput. Biol.</em> 13 (1): e1005331. <a href="https://doi.org/10.1371/journal.pcbi.1005331" class="uri">https://doi.org/10.1371/journal.pcbi.1005331</a>.</div></div><div id="refs" class="references"><div id="ref-FroehlichKal2017"> </div> <div id="ref-FroehlichThe2016"> </div></div><div id="refs" class="references"><div id="ref-FroehlichThe2016">Fröhlich, F., F. J. Theis, J. O. Rädler, and J. Hasenauer. 2017. “Parameter Estimation for Dynamical Systems with Discrete Events and Logical Operations.” <em>Bioinformatics</em> 33 (7): 1049–56. <a href="https://doi.org/10.1093/bioinformatics/btw764" class="uri">https://doi.org/10.1093/bioinformatics/btw764</a>.</div></div><div id="refs" class="references"><div id="ref-FroehlichThe2016"> </div> <div id="ref-KazeroonianThe2017"> </div></div><div id="refs" class="references"><div id="ref-KazeroonianThe2017">Kazeroonian, A., F. J. Theis, and J. Hasenauer. 2017. “A Scalable Moment-Closure Approximation for Large-Scale Biochemical Reaction Networks.” <em>Bioinformatics</em> 33 (14): i293–i300. <a href="https://doi.org/10.1093/bioinformatics/btx249" class="uri">https://doi.org/10.1093/bioinformatics/btx249</a>.</div></div><div id="refs" class="references"><div id="ref-KazeroonianThe2017"> </div> <div id="ref-MaierLoo2017"> </div></div><div id="refs" class="references"><div id="ref-MaierLoo2017">Maier, C., C. Loos, and J. Hasenauer. 2017. “Robust Parameter Estimation for Dynamical Systems from Outlier-Corrupted Data.” <em>Bioinformatics</em> 33 (5): 718–25. <a href="https://doi.org/10.1093/bioinformatics/btw703" class="uri">https://doi.org/10.1093/bioinformatics/btw703</a>.</div></div><div id="refs" class="references"><div id="ref-MaierLoo2017"> </div> </div> <h1 id="section" class="unnumbered">2016</h1>
<div id="refs" class="references"> <div id="ref-BoigerHas2016"> </div></div><div id="refs" class="references"><div id="ref-BoigerHas2016">Boiger, R., J. Hasenauer, S. Hross, and B. Kaltenbacher. 2016. “Integration Based Profile Likelihood Calculation for PDE Constrained Parameter Estimation Problems.” <em>Inverse Prob.</em> 32 (12): 125009. <a href="https://doi.org/10.1088/0266-5611/32/12/125009" class="uri">https://doi.org/10.1088/0266-5611/32/12/125009</a>.</div></div><div id="refs" class="references"><div id="ref-BoigerHas2016"> </div> <div id="ref-FiedlerRae2016"> </div></div><div id="refs" class="references"><div id="ref-FiedlerRae2016">Fiedler, A., S. Raeth, F. J. Theis, A. Hausser, and J. Hasenauer. 2016. “Tailored Parameter Optimization Methods for Ordinary Differential Equation Models with Steady-State Constraints.” <em>BMC Syst. Biol.</em> 10 (80). <a href="https://doi.org/10.1186/s12918-016-0319-7" class="uri">https://doi.org/10.1186/s12918-016-0319-7</a>.</div></div><div id="refs" class="references"><div id="ref-FiedlerRae2016"> </div> <div id="ref-FroehlichTho2016"> </div></div><div id="refs" class="references"><div id="ref-FroehlichTho2016">Fröhlich, F., P. Thomas, A. Kazeroonian, F. J. Theis, R. Grima, and J. Hasenauer. 2016. “Inference for Stochastic Chemical Kinetics Using Moment Equations and System Size Expansion.” <em>PLoS Comput. Biol.</em> 12 (7): e1005030. <a href="https://doi.org/10.1371/journal.pcbi.1005030" class="uri">https://doi.org/10.1371/journal.pcbi.1005030</a>.</div></div><div id="refs" class="references"><div id="ref-FroehlichTho2016"> </div> <div id="ref-HrossFie2016"> </div></div><div id="refs" class="references"><div id="ref-HrossFie2016">Hross, S., A. Fiedler, F. J. Theis, and J. Hasenauer. 2016. “Quantitative Comparison of Competing PDE Models for Pom1p Dynamics in Fission Yeast.” In <em>Proc. 6th IFAC Conf. Found. Syst. Biol. Eng.</em>, edited by R. Findeisen, E. Bullinger, E. Balsa-Canto, and K. Bernaerts, 49:264–69. 26. IFAC-PapersOnLine. <a href="https://doi.org/10.1016/j.ifacol.2016.12.136" class="uri">https://doi.org/10.1016/j.ifacol.2016.12.136</a>.</div></div><div id="refs" class="references"><div id="ref-HrossFie2016"> </div> <div id="ref-KazeroonianFro2016"> </div></div><div id="refs" class="references"><div id="ref-KazeroonianFro2016">Kazeroonian, A., F. Fröhlich, A. Raue, F. J. Theis, and J. Hasenauer. 2016. “CERENA: Chemical REaction Network Analyzer – A Toolbox for the Simulation and Analysis of Stochastic Chemical Kinetics.” <em>PLoS ONE</em> 11 (1): e0146732. <a href="https://doi.org/10.1371/journal.pone.0146732" class="uri">https://doi.org/10.1371/journal.pone.0146732</a>.</div></div><div id="refs" class="references"><div id="ref-KazeroonianFro2016"> </div> <div id="ref-LoosFie2016"> </div></div><div id="refs" class="references"><div id="ref-LoosFie2016">Loos, C., A. Fiedler, and J. Hasenauer. 2016. “Parameter Estimation for Reaction Rate Equation Constrained Mixture Models.” In <em>Proc. 13th Int. Conf. Comp. Meth. Syst. Biol.</em>, edited by E. Bartocci, P. Lio, and N. Paoletti, 186–200. Lecture Notes in Bioinformatics. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-45177-0" class="uri">https://doi.org/10.1007/978-3-319-45177-0</a>.</div></div><div id="refs" class="references"><div id="ref-LoosFie2016"> </div> </div> <h1 id="section" class="unnumbered">2015</h1>
<div id="refs" class="references"> <div id="ref-LoosMar2015"> </div></div><div id="refs" class="references"><div id="ref-LoosMar2015">Loos, C., C. Marr, F. J. Theis, and J. Hasenauer. 2015. “Computational Methods in Systems Biology.” In, edited by O. Roux and J. Bourdon, 9308:52–63. Lecture Notes in Computer Science. Springer International Publishing.</div></div><div id="refs" class="references"><div id="ref-LoosMar2015"> </div> </div> </div></div><!-- contents -->
</div><!-- PageDoc -->
</div><!-- doc-content -->
<!-- start footer part -->
<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
  <ul>
    <li class="footer">Generated on Sun Jul 19 2020 20:54:20 for AMICI by
    <a href="http://www.doxygen.org/index.html">
    <img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.8.18 </li>
  </ul>
</div>
</body>
</html>
