List of publications using A\+M\+I\+CI. Total number is 35.

If you applied A\+M\+I\+CI in your work and your publication is missing, please let us know via a new Github issue.

\doxysubsection*{2019}

  

Dharmarajan, Lekshmi, Hans-\/\+Michael Kaltenbach, Fabian Rudolf, and Joerg Stelling. 2019. “A Simple and Flexible Computational Framework for Inferring Sources of Heterogeneity from Single-\/\+Cell Dynamics.\+” {\itshape Cell Systems} 8 (1). Elsevier\+: 15–26.\+e11. \href{https://doi.org/10.1016/j.cels.2018.12.007}{\texttt{ https\+://doi.\+org/10.\+1016/j.\+cels.\+2018.\+12.\+007}}.

   

Fischer, David S., Anna K. Fiedler, Eric Kernfeld, Ryan M. J. Genga, Aimée Bastidas-\/\+Ponce, Mostafa Bakhti, Heiko Lickert, Jan Hasenauer, Rene Maehr, and Fabian J. Theis. 2019. “\+Inferring Population Dynamics from Single-\/\+Cell Rna-\/\+Sequencing Time Series Data.\+” {\itshape Nature Biotechnology} 37\+: 461–68. \href{https://doi.org/10.1038/s41587-019-0088-0}{\texttt{ https\+://doi.\+org/10.\+1038/s41587-\/019-\/0088-\/0}}.

   

Gregg, Robert W, Saumendra N Sarkar, and Jason E Shoemaker. 2019. “\+Mathematical Modeling of the c\+G\+AS Pathway Reveals Robustness of Dna Sensing to Trex1 Feedback.\+” {\itshape Journal of Theoretical Biology} 462 (February)\+: 148–57. \href{https://doi.org/10.1016/j.jtbi.2018.11.001}{\texttt{ https\+://doi.\+org/10.\+1016/j.\+jtbi.\+2018.\+11.\+001}}.

   

Kapfer, Eva-\/\+Maria, Paul Stapor, and Jan Hasenauer. 2019. “\+Challenges in the Calibration of Large-\/\+Scale Ordinary Differential Equation Models.\+” {\itshape Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (F\+O\+S\+BE)}.

   

Lines, Glenn Terje, Lukasz Paszkowski, Leonard Schmiester, Daniel Weindl, Paul Stapor, and Jan Hasenauer. 2019. “\+Efficient Computation of Steady States in Large-\/\+Scale Ode Models of Biochemical Reaction Networks.\+” {\itshape Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (F\+O\+S\+BE)}.

   

Nousiainen, Kari, Jukka Intosalmi, and Harri Lähdesmäki. 2019. “A Mathematical Model for Enhancer Activation Kinetics During Cell Differentiation.\+” In {\itshape Algorithms for Computational Biology}, edited by Ian Holmes, Carlos Martín-\/\+Vide, and Miguel A. Vega-\/\+Rodríguez, 191–202. Cham\+: Springer International Publishing.

   

Pedretscher, B., B. Kaltenbacher, and O. Pfeiler. 2019. “\+Parameter Identification and Uncertainty Quantification in Stochastic State Space Models and Its Application to Texture Analysis.\+” {\itshape Applied Numerical Mathematics} 146\+: 38–54. \href{https://doi.org/https://doi.org/10.1016/j.apnum.2019.06.020}{\texttt{ https\+://doi.\+org/https\+://doi.\+org/10.\+1016/j.\+apnum.\+2019.\+06.\+020}}.

   

Pitt, Jake Alan, and Julio R Banga. 2019. “\+Parameter Estimation in Models of Biological Oscillators\+: An Automated Regularised Estimation Approach.\+” {\itshape B\+MC Bioinformatics} 20 (1)\+: 82. \href{https://doi.org/10.1186/s12859-019-2630-y}{\texttt{ https\+://doi.\+org/10.\+1186/s12859-\/019-\/2630-\/y}}.

   

Schmiester, Leonard, Yannik Schälte, Fabian Fröhlich, Jan Hasenauer, and Daniel Weindl. 2019. “\+Efficient parameterization of large-\/scale dynamic models based on relative measurements.\+” {\itshape Bioinformatics}, July. \href{https://doi.org/10.1093/bioinformatics/btz581}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/btz581}}.

   

Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2019. “\+Statistical Inference of Mechanistic Models from Qualitative Data Using an Efficient Optimal Scaling Approach.\+” {\itshape bio\+Rxiv}. Cold Spring Harbor Laboratory. \href{https://doi.org/10.1101/848648}{\texttt{ https\+://doi.\+org/10.\+1101/848648}}.

   

Stapor, Paul, Leonard Schmiester, Christoph Wierling, Bodo Lange, Daniel Weindl, and Jan Hasenauer. 2019. “\+Mini-\/\+Batch Optimization Enables Training of Ode Models on Large-\/\+Scale Datasets.\+” {\itshape bio\+Rxiv}. Cold Spring Harbor Laboratory. \href{https://doi.org/10.1101/859884}{\texttt{ https\+://doi.\+org/10.\+1101/859884}}.

   

Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R. Banga. 2019. “A Comparison of Methods for Quantifying Prediction Uncertainty in Systems Biology.\+” {\itshape Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (F\+O\+S\+BE)}.

   

Wang, Dantong, Paul Stapor, and Jan Hasenauer. 2019. “\+Dirac Mixture Distributions for the Approximation of Mixed Effects Models.\+” {\itshape Accepted for Publication in Proc. Of the Foundations of Syst. Biol. In Engin. (F\+O\+S\+BE)}.

   \doxysubsection*{2018}

  

Ballnus, Benjamin, Steffen Schaper, Fabian J Theis, and Jan Hasenauer. 2018. “\+Bayesian Parameter Estimation for Biochemical Reaction Networks Using Region-\/\+Based Adaptive Parallel Tempering.\+” {\itshape Bioinformatics} 34 (13)\+: i494–i501. \href{https://doi.org/10.1093/bioinformatics/bty229}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/bty229}}.

   

Bast, Lisa, Filippo Calzolari, Michael Strasser, Jan Hasenauer, Fabian J. Theis, Jovica Ninkovic, and Carsten Marr. 2018. “\+Subtle Changes in Clonal Dynamics Underlie the Age-\/\+Related Decline in Neurogenesis.\+” {\itshape Cell Reports}.

   

Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “\+Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-\/\+Cancer Pathway Model.\+” {\itshape Cell Systems} 7 (6). Elsevier\+: 567–579.\+e6. \href{https://doi.org/10.1016/j.cels.2018.10.013}{\texttt{ https\+://doi.\+org/10.\+1016/j.\+cels.\+2018.\+10.\+013}}.

   

Fröhlich, Fabian, Anita Reiser, Laura Fink, Daniel Woschée, Thomas Ligon, Fabian Joachim Theis, Joachim Oskar Rädler, and Jan Hasenauer. 2018. “\+Multi-\/\+Experiment Nonlinear Mixed Effect Modeling of Single-\/\+Cell Translation Kinetics After Transfection.\+” {\itshape Npj Systems Biology and Applications} 5 (1)\+: 1. \href{https://doi.org/10.1038/s41540-018-0079-7}{\texttt{ https\+://doi.\+org/10.\+1038/s41540-\/018-\/0079-\/7}}.

   

Kaltenbacher, Barbara, and Barbara Pedretscher. 2018. “\+Parameter Estimation in Sdes via the Fokker–\+Planck Equation\+: Likelihood Function and Adjoint Based Gradient Computation.\+” {\itshape Journal of Mathematical Analysis and Applications} 465 (2)\+: 872–84. \href{https://doi.org/https://doi.org/10.1016/j.jmaa.2018.05.048}{\texttt{ https\+://doi.\+org/https\+://doi.\+org/10.\+1016/j.\+jmaa.\+2018.\+05.\+048}}.

   

Loos, Carolin, Sabrina Krause, and Jan Hasenauer. 2018. “\+Hierarchical Optimization for the Efficient Parametrization of O\+DE Models.\+” {\itshape Bioinformatics} 34 (24)\+: 4266–73. \href{https://doi.org/10.1093/bioinformatics/bty514}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/bty514}}.

   

Loos, Carolin, Katharina Moeller, Fabian Fröhlich, Tim Hucho, and Jan Hasenauer. 2018. “A Hierarchical, Data-\/\+Driven Approach to Modeling Single-\/\+Cell Populations Predicts Latent Causes of Cell-\/to-\/\+Cell Variability.\+” {\itshape Cell Systems} 6 (5). Elsevier\+: 593–603. \href{https://doi.org/10.1016/j.cels.2018.04.008}{\texttt{ https\+://doi.\+org/10.\+1016/j.\+cels.\+2018.\+04.\+008}}.

   

Schälte, Y., P. Stapor, and J. Hasenauer. 2018. “\+Evaluation of Derivative-\/\+Free Optimizers for Parameter Estimation in Systems Biology.\+” {\itshape F\+A\+C-\/\+Papers\+On\+Line} 51 (19)\+: 98–101.

   

Stapor, Paul, Fabian Fröhlich, and Jan Hasenauer. 2018. “\+Optimization and Profile Calculation of O\+DE Models Using Second Order Adjoint Sensitivity Analysis.\+” {\itshape Bioinformatics} 34 (13). Oxford University Press\+: i151–i159.

   

Villaverde, Alejandro F., Fabian Froehlich, Daniel Weindl, Jan Hasenauer, and Julio R Banga. 2018. “\+Benchmarking Optimization Methods for Parameter Estimation in Large Kinetic Models.\+” {\itshape Bioinformatics}, bty736.

   \doxysubsection*{2017}

  

Ballnus, B., S. Hug, K. Hatz, L. Görlitz, J. Hasenauer, and F. J. Theis. 2017. “\+Comprehensive Benchmarking of Markov Chain Monte Carlo Methods for Dynamical Systems.\+” {\itshape B\+MC Syst. Biol.} 11 (63). \href{https://doi.org/10.1186/s12918-017-0433-1}{\texttt{ https\+://doi.\+org/10.\+1186/s12918-\/017-\/0433-\/1}}.

   

Fröhlich, F., B. Kaltenbacher, F. J. Theis, and J. Hasenauer. 2017. “\+Scalable Parameter Estimation for Genome-\/\+Scale Biochemical Reaction Networks.\+” {\itshape P\+LoS Comput. Biol.} 13 (1)\+: e1005331. \href{https://doi.org/10.1371/journal.pcbi.1005331}{\texttt{ https\+://doi.\+org/10.\+1371/journal.\+pcbi.\+1005331}}.

   

Fröhlich, F., F. J. Theis, J. O. Rädler, and J. Hasenauer. 2017. “\+Parameter Estimation for Dynamical Systems with Discrete Events and Logical Operations.\+” {\itshape Bioinformatics} 33 (7)\+: 1049–56. \href{https://doi.org/10.1093/bioinformatics/btw764}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/btw764}}.

   

Kazeroonian, A., F. J. Theis, and J. Hasenauer. 2017. “A Scalable Moment-\/\+Closure Approximation for Large-\/\+Scale Biochemical Reaction Networks.\+” {\itshape Bioinformatics} 33 (14)\+: i293–i300. \href{https://doi.org/10.1093/bioinformatics/btx249}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/btx249}}.

   

Maier, C., C. Loos, and J. Hasenauer. 2017. “\+Robust Parameter Estimation for Dynamical Systems from Outlier-\/\+Corrupted Data.\+” {\itshape Bioinformatics} 33 (5)\+: 718–25. \href{https://doi.org/10.1093/bioinformatics/btw703}{\texttt{ https\+://doi.\+org/10.\+1093/bioinformatics/btw703}}.

   \doxysubsection*{2016}

  

Boiger, R., J. Hasenauer, S. Hross, and B. Kaltenbacher. 2016. “\+Integration Based Profile Likelihood Calculation for P\+DE Constrained Parameter Estimation Problems.\+” {\itshape Inverse Prob.} 32 (12)\+: 125009. \href{https://doi.org/10.1088/0266-5611/32/12/125009}{\texttt{ https\+://doi.\+org/10.\+1088/0266-\/5611/32/12/125009}}.

   

Fiedler, A., S. Raeth, F. J. Theis, A. Hausser, and J. Hasenauer. 2016. “\+Tailored Parameter Optimization Methods for Ordinary Differential Equation Models with Steady-\/\+State Constraints.\+” {\itshape B\+MC Syst. Biol.} 10 (80). \href{https://doi.org/10.1186/s12918-016-0319-7}{\texttt{ https\+://doi.\+org/10.\+1186/s12918-\/016-\/0319-\/7}}.

   

Fröhlich, F., P. Thomas, A. Kazeroonian, F. J. Theis, R. Grima, and J. Hasenauer. 2016. “\+Inference for Stochastic Chemical Kinetics Using Moment Equations and System Size Expansion.\+” {\itshape P\+LoS Comput. Biol.} 12 (7)\+: e1005030. \href{https://doi.org/10.1371/journal.pcbi.1005030}{\texttt{ https\+://doi.\+org/10.\+1371/journal.\+pcbi.\+1005030}}.

   

Hross, S., A. Fiedler, F. J. Theis, and J. Hasenauer. 2016. “\+Quantitative Comparison of Competing P\+DE Models for Pom1p Dynamics in Fission Yeast.\+” In {\itshape Proc. 6th I\+F\+AC Conf. Found. Syst. Biol. Eng.}, edited by R. Findeisen, E. Bullinger, E. Balsa-\/\+Canto, and K. Bernaerts, 49\+:264–69. 26. I\+F\+A\+C-\/\+Papers\+On\+Line. \href{https://doi.org/10.1016/j.ifacol.2016.12.136}{\texttt{ https\+://doi.\+org/10.\+1016/j.\+ifacol.\+2016.\+12.\+136}}.

   

Kazeroonian, A., F. Fröhlich, A. Raue, F. J. Theis, and J. Hasenauer. 2016. “\+C\+E\+R\+E\+NA\+: Chemical R\+Eaction Network Analyzer – A Toolbox for the Simulation and Analysis of Stochastic Chemical Kinetics.\+” {\itshape P\+LoS O\+NE} 11 (1)\+: e0146732. \href{https://doi.org/10.1371/journal.pone.0146732}{\texttt{ https\+://doi.\+org/10.\+1371/journal.\+pone.\+0146732}}.

   

Loos, C., A. Fiedler, and J. Hasenauer. 2016. “\+Parameter Estimation for Reaction Rate Equation Constrained Mixture Models.\+” In {\itshape Proc. 13th Int. Conf. Comp. Meth. Syst. Biol.}, edited by E. Bartocci, P. Lio, and N. Paoletti, 186–200. Lecture Notes in Bioinformatics. Springer International Publishing. \href{https://doi.org/10.1007/978-3-319-45177-0}{\texttt{ https\+://doi.\+org/10.\+1007/978-\/3-\/319-\/45177-\/0}}.

   \doxysubsection*{2015}

  

Loos, C., C. Marr, F. J. Theis, and J. Hasenauer. 2015. “\+Computational Methods in Systems Biology.\+” In, edited by O. Roux and J. Bourdon, 9308\+:52–63. Lecture Notes in Computer Science. Springer International Publishing.

   