diff --git a/c:/Users/sharpie/Downloads/HDF5-1.8.7_win_x86/include/H5public.h b/c:/swarm/include/H5public.h
index fc7c570..c55a9d7 100644
--- a/include/H5public.h
+++ b/include/H5public.h
@@ -125,6 +125,11 @@ typedef int herr_t;
 typedef unsigned int hbool_t;
 typedef int htri_t;
 
+
+/* Contrary to popular belief, MinGW **does** define ssize_t */
+#ifdef __MINGW32__
+#  define H5_SIZEOF_SSIZE_T H5_SIZEOF_LONG
+#else
 /* Define the ssize_t type if it not is defined */
 #if H5_SIZEOF_SSIZE_T==0
 /* Undefine this size, we will re-define it in one of the sections below */
@@ -142,6 +147,7 @@ typedef long long ssize_t;
 #   error "nothing appropriate for ssize_t"
 #endif
 #endif
+#endif
 
 /*
  * The sizes of file objects have their own types defined here, use a 64-bit
