# References

List of publications using AMICI. Total number is 64.

If you applied AMICI in your work and your publication is missing, please let us know via a new Github issue.

<h1 class="unnumbered" id="section">2022</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-SchmuckerFar2022">
<p>Schmucker, Robin, Gabriele Farina, James Faeder, Fabian Fröhlich, Ali Sinan Saglam, and Tuomas Sandholm. 2022. “Combination Treatment Optimization Using a Pan-Cancer Pathway Model.” <em>PLOS Computational Biology</em> 17 (12): 1–22. <a href="https://doi.org/10.1371/journal.pcbi.1009689">https://doi.org/10.1371/journal.pcbi.1009689</a>.</p>
</div>
<div id="ref-StaporSch2022">
<p>Stapor, Paul, Leonard Schmiester, Christoph Wierling, Simon Merkt, Dilan Pathirana, Bodo M. H. Lange, Daniel Weindl, and Jan Hasenauer. 2022. “Mini-batch optimization enables training of ODE models on large-scale datasets.” <em>Nature Communications</em> 13 (1): 34. <a href="https://doi.org/10.1038/s41467-021-27374-6">https://doi.org/10.1038/s41467-021-27374-6</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2021</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-AdlungSta2021">
<p>Adlung, Lorenz, Paul Stapor, Christian Tönsing, Leonard Schmiester, Luisa E. Schwarzmüller, Lena Postawa, Dantong Wang, et al. 2021. “Cell-to-Cell Variability in Jak2/Stat5 Pathway Components and Cytoplasmic Volumes Defines Survival Threshold in Erythroid Progenitor Cells.” <em>Cell Reports</em> 36 (6): 109507. <a href="https://doi.org/10.1016/j.celrep.2021.109507">https://doi.org/10.1016/j.celrep.2021.109507</a>.</p>
</div>
<div id="ref-BastBuc2021">
<p>Bast, Lisa, Michèle C. Buck, Judith S. Hecker, Robert A. J. Oostendorp, Katharina S. Götze, and Carsten Marr. 2021. “Computational Modeling of Stem and Progenitor Cell Kinetics Identifies Plausible Hematopoietic Lineage Hierarchies.” <em>iScience</em> 24 (2): 102120. <a href="https://doi.org/10.1016/j.isci.2021.102120">https://doi.org/10.1016/j.isci.2021.102120</a>.</p>
</div>
<div id="ref-Contento2021.10.01.21263052">
<p>Contento, Lorenzo, Noemi Castelletti, Elba Raimúndez, Ronan Le Gleut, Yannik Schälte, Paul Stapor, Ludwig Christian Hinske, et al. 2021. “Integrative Modelling of Reported Case Numbers and Seroprevalence Reveals Time-Dependent Test Efficiency and Infection Rates.” <em>medRxiv</em>. <a href="https://doi.org/10.1101/2021.10.01.21263052">https://doi.org/10.1101/2021.10.01.21263052</a>.</p>
</div>
<div id="ref-ErdemMut2021">
<p>Erdem, Cemal, Arnab Mutsuddy, Ethan M. Bensman, William B. Dodd, Michael M. Saint-Antoine, Mehdi Bouhaddou, Robert C. Blake, et al. 2021. “A Scalable, Open-Source Implementation of a Large-Scale Mechanistic Model for Single Cell Proliferation and Death Signaling.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.11.09.373407">https://doi.org/10.1101/2020.11.09.373407</a>.</p>
</div>
<div id="ref-Froehlich2021.05.20.445065">
<p>Fröhlich, Fabian, and Peter K. Sorger. 2021. “Fides: Reliable Trust-Region Optimization for Parameter Estimation of Ordinary Differential Equation Models.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.05.20.445065">https://doi.org/10.1101/2021.05.20.445065</a>.</p>
</div>
<div id="ref-Gaspari2021">
<p>Gaspari, Erika. 2021. “Model-Driven Design of Mycoplasma as a Vaccine Chassis.” PhD thesis, Wageningen: Wageningen University. <a href="https://doi.org/10.18174/539593">https://doi.org/10.18174/539593</a>.</p>
</div>
<div id="ref-GudinaAli2021">
<p>Gudina, Esayas Kebede, Solomon Ali, Eyob Girma, Addisu Gize, Birhanemeskel Tegene, Gadissa Bedada Hundie, Wondewosen Tsegaye Sime, et al. 2021. “Seroepidemiology and model-based prediction of SARS-CoV-2 in Ethiopia: longitudinal cohort study among front-line hospital workers and communities.” <em>The Lancet Global Health</em> 9 (11): e1517–e1527. <a href="https://doi.org/10.1016/S2214-109X(21)00386-7">https://doi.org/10.1016/S2214-109X(21)00386-7</a>.</p>
</div>
<div id="ref-Maier2021">
<p>Maier, Corinna. 2021. “Bayesian Data Assimilation and Reinforcement Learning for Model-Informed Precision Dosing in Oncology.” Doctoralthesis, Universität Potsdam. <a href="https://doi.org/10.25932/publishup-51587">https://doi.org/10.25932/publishup-51587</a>.</p>
</div>
<div id="ref-RaimundezDud2021">
<p>Raimúndez, Elba, Erika Dudkin, Jakob Vanhoefer, Emad Alamoudi, Simon Merkt, Lara Fuhrmann, Fan Bai, and Jan Hasenauer. 2021. “COVID-19 Outbreak in Wuhan Demonstrates the Limitations of Publicly Available Case Numbers for Epidemiological Modeling.” <em>Epidemics</em> 34: 100439. <a href="https://doi.org/10.1016/j.epidem.2021.100439">https://doi.org/10.1016/j.epidem.2021.100439</a>.</p>
</div>
<div id="ref-Schmiester2021.02.06.430039">
<p>Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2021. “Efficient Gradient-Based Parameter Estimation for Dynamic Models Using Qualitative Data.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.02.06.430039">https://doi.org/10.1101/2021.02.06.430039</a>.</p>
</div>
<div id="ref-StaedterSch2021">
<p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2021. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>Scientific Reports</em> 11 (1): 2696. <a href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.</p>
</div>
<div id="ref-StenPod2021">
<p>Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder, Maria Engström, and Gunnar Cedersund. 2021. “A Multi-Data Based Quantitative Model for the Neurovascular Coupling in the Brain.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.</p>
</div>
<div id="ref-TomasoniPar2021">
<p>Tomasoni, Danilo, Alessio Paris, Stefano Giampiccolo, Federico Reali, Giulia Simoni, Luca Marchetti, Chanchala Kaddi, et al. 2021. “QSPcc Reduces Bottlenecks in Computational Model Simulations.” <em>Communications Biology</em> 4 (1): 1022. <a href="https://doi.org/10.1038/s42003-021-02553-9">https://doi.org/10.1038/s42003-021-02553-9</a>.</p>
</div>
<div id="ref-VanhoeferMat2021">
<p>Vanhoefer, Jakob, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte, and Jan Hasenauer. 2021. “Yaml2sbml: Human-Readable and -Writable Specification of Ode Models and Their Conversion to Sbml.” <em>Journal of Open Source Software</em> 6 (61): 3215. <a href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.</p>
</div>
<div id="ref-vanRosmalenSmi2021">
<p>van Rosmalen, R. P., R. W. Smith, V. A. P. Martins dos Santos, C. Fleck, and M. Suarez-Diez. 2021. “Model Reduction of Genome-Scale Metabolic Models as a Basis for Targeted Kinetic Models.” <em>Metabolic Engineering</em> 64: 74–84. <a href="https://doi.org/10.1016/j.ymben.2021.01.008">https://doi.org/10.1016/j.ymben.2021.01.008</a>.</p>
</div>
<div id="ref-VillaverdePat2021">
<p>Villaverde, Alejandro F, Dilan Pathirana, Fabian Fröhlich, Jan Hasenauer, and Julio R Banga. 2021. “A protocol for dynamic model calibration.” <em>Briefings in Bioinformatics</em>, October. <a href="https://doi.org/10.1093/bib/bbab387">https://doi.org/10.1093/bib/bbab387</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2020</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-AlabertLoo2020">
<p>Alabert, Constance, Carolin Loos, Moritz Voelker-Albert, Simona Graziano, Ignasi Forné, Nazaret Reveron-Gomez, Lea Schuh, et al. 2020. “Domain Model Explains Propagation Dynamics and Stability of Histone H3k27 and H3k36 Methylation Landscapes.” <em>Cell Reports</em> 30 (4): 1223–1234.e8. <a href="https://doi.org/10.1016/j.celrep.2019.12.060">https://doi.org/10.1016/j.celrep.2019.12.060</a>.</p>
</div>
<div id="ref-ErdemBen2020">
<p>Erdem, Cemal, Ethan M. Bensman, Arnab Mutsuddy, Michael M. Saint-Antoine, Mehdi Bouhaddou, Robert C. Blake, Will Dodd, et al. 2020. “A Simple and Efficient Pipeline for Construction, Merging, Expansion, and Simulation of Large-Scale, Single-Cell Mechanistic Models.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.11.09.373407">https://doi.org/10.1101/2020.11.09.373407</a>.</p>
</div>
<div id="ref-GerosaChi2020">
<p>Gerosa, Luca, Christopher Chidley, Fabian Fröhlich, Gabriela Sanchez, Sang Kyun Lim, Jeremy Muhlich, Jia-Yun Chen, et al. 2020. “Receptor-Driven Erk Pulses Reconfigure Mapk Signaling and Enable Persistence of Drug-Adapted Braf-Mutant Melanoma Cells.” <em>Cell Systems</em>. <a href="https://doi.org/10.1016/j.cels.2020.10.002">https://doi.org/10.1016/j.cels.2020.10.002</a>.</p>
</div>
<div id="ref-Kuritz2020.03.30.015909">
<p>Kuritz, Karsten, Alain R Bonny, João Pedro Fonseca, and Frank Allgöwer. 2020. “PDE-Constrained Optimization for Estimating Population Dynamics over Cell Cycle from Static Single Cell Measurements.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.03.30.015909">https://doi.org/10.1101/2020.03.30.015909</a>.</p>
</div>
<div id="ref-MaierHar2020">
<p>Maier, Corinna, Niklas Hartung, Charlotte Kloft, Wilhelm Huisinga, and Jana de Wiljes. 2020. “Reinforcement Learning and Bayesian Data Assimilation for Model-Informed Precision Dosing in Oncology.” <a href="http://arxiv.org/abs/2006.01061">http://arxiv.org/abs/2006.01061</a>.</p>
</div>
<div id="ref-SchaelteHas2020">
<p>Schälte, Yannik, and Jan Hasenauer. 2020. “Efficient exact inference for dynamical systems with noisy measurements using sequential approximate Bayesian computation.” <em>Bioinformatics</em> 36 (July): i551–i559. <a href="https://doi.org/10.1093/bioinformatics/btaa397">https://doi.org/10.1093/bioinformatics/btaa397</a>.</p>
</div>
<div id="ref-SchmiesterWei2020">
<p>Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2020. “Parameterization of Mechanistic Models from Qualitative Data Using an Efficient Optimal Scaling Approach.” <em>Journal of Mathematical Biology</em>, July. <a href="https://doi.org/10.1007/s00285-020-01522-w">https://doi.org/10.1007/s00285-020-01522-w</a>.</p>
</div>
<div id="ref-SchuhLoo2020">
<p>Schuh, Lea, Carolin Loos, Daniil Pokrovsky, Axel Imhof, Ralph A. W. Rupp, and Carsten Marr. 2020. “H4K20 Methylation Is Differently Regulated by Dilution and Demethylation in Proliferating and Cell-Cycle-Arrested Xenopus Embryos.” <em>Cell Systems</em> 11 (6): 653–662.e8. <a href="https://doi.org/10.1016/j.cels.2020.11.003">https://doi.org/10.1016/j.cels.2020.11.003</a>.</p>
</div>
<div id="ref-Sten2020">
<p>Sten, Sebastian. 2020. “Mathematical Modeling of Neurovascular Coupling.” Linköping University Medical Dissertations. PhD thesis, Linköping UniversityLinköping UniversityLinköping University, Division of Diagnostics; Specialist Medicine, Faculty of Medicine; Health Sciences, Center for Medical Image Science; Visualization (CMIV); Linköping University, Division of Diagnostics; Specialist Medicine. <a href="https://doi.org/10.3384/diss.diva-167806">https://doi.org/10.3384/diss.diva-167806</a>.</p>
</div>
<div id="ref-StenEli2020">
<p>Sten, Sebastian, Fredrik Elinder, Gunnar Cedersund, and Maria Engström. 2020. “A Quantitative Analysis of Cell-Specific Contributions and the Role of Anesthetics to the Neurovascular Coupling.” <em>NeuroImage</em> 215: 116827. <a href="https://doi.org/10.1016/j.neuroimage.2020.116827">https://doi.org/10.1016/j.neuroimage.2020.116827</a>.</p>
</div>
<div id="ref-TsipaPit2020">
<p>Tsipa, Argyro, Jake Alan Pitt, Julio R. Banga, and Athanasios Mantalaris. 2020. “A Dual-Parameter Identification Approach for Data-Based Predictive Modeling of Hybrid Gene Regulatory Network-Growth Kinetics in Pseudomonas Putida Mt-2.” <em>Bioprocess and Biosystems Engineering</em> 43 (9): 1671–88. <a href="https://doi.org/10.1007/s00449-020-02360-2">https://doi.org/10.1007/s00449-020-02360-2</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2019</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-DharmarajanKal2019">
<p>Dharmarajan, Lekshmi, Hans-Michael Kaltenbach, Fabian Rudolf, and Joerg Stelling. 2019. “A Simple and Flexible Computational Framework for Inferring Sources of Heterogeneity from Single-Cell Dynamics.” <em>Cell Systems</em> 8 (1): 15–26.e11. <a href="https://doi.org/10.1016/j.cels.2018.12.007">https://doi.org/10.1016/j.cels.2018.12.007</a>.</p>
</div>
<div id="ref-FischerFie2017">
<p>Fischer, David S., Anna K. Fiedler, Eric Kernfeld, Ryan M. J. Genga, Aimée Bastidas-Ponce, Mostafa Bakhti, Heiko Lickert, Jan Hasenauer, Rene Maehr, and Fabian J. Theis. 2019. “Inferring Population Dynamics from Single-Cell Rna-Sequencing Time Series Data.” <em>Nature Biotechnology</em> 37: 461–68. <a href="https://doi.org/10.1038/s41587-019-0088-0">https://doi.org/10.1038/s41587-019-0088-0</a>.</p>
</div>
<div id="ref-GreggSar2019">
<p>Gregg, Robert W, Saumendra N Sarkar, and Jason E Shoemaker. 2019. “Mathematical Modeling of the cGAS Pathway Reveals Robustness of Dna Sensing to Trex1 Feedback.” <em>Journal of Theoretical Biology</em> 462 (February): 148–57. <a href="https://doi.org/10.1016/j.jtbi.2018.11.001">https://doi.org/10.1016/j.jtbi.2018.11.001</a>.</p>
</div>
<div id="ref-KapferSta2019">
<p>Kapfer, Eva-Maria, Paul Stapor, and Jan Hasenauer. 2019. “Challenges in the Calibration of Large-Scale Ordinary Differential Equation Models.” <em>IFAC-PapersOnLine</em> 52 (26): 58–64. <a href="https://doi.org/10.1016/j.ifacol.2019.12.236">https://doi.org/10.1016/j.ifacol.2019.12.236</a>.</p>
</div>
<div id="ref-NousiainenInt2019">
<p>Nousiainen, Kari, Jukka Intosalmi, and Harri Lähdesmäki. 2019. “A Mathematical Model for Enhancer Activation Kinetics During Cell Differentiation.” In <em>Algorithms for Computational Biology</em>, edited by Ian Holmes, Carlos Martı́n-Vide, and Miguel A. Vega-Rodrı́guez, 191–202. Cham: Springer International Publishing. <a href="https://doi.org/10.1007/978-3-030-18174-1_14">https://doi.org/10.1007/978-3-030-18174-1_14</a>.</p>
</div>
<div id="ref-PedretscherKal2019">
<p>Pedretscher, B., B. Kaltenbacher, and O. Pfeiler. 2019. “Parameter Identification and Uncertainty Quantification in Stochastic State Space Models and Its Application to Texture Analysis.” <em>Applied Numerical Mathematics</em> 146: 38–54. <a href="https://doi.org/10.1016/j.apnum.2019.06.020">https://doi.org/10.1016/j.apnum.2019.06.020</a>.</p>
</div>
<div id="ref-PittBan2019">
<p>Pitt, Jake Alan, and Julio R Banga. 2019. “Parameter Estimation in Models of Biological Oscillators: An Automated Regularised Estimation Approach.” <em>BMC Bioinformatics</em> 20 (1): 82. <a href="https://doi.org/10.1186/s12859-019-2630-y">https://doi.org/10.1186/s12859-019-2630-y</a>.</p>
</div>
<div id="ref-SchmiesterSch2019">
<p>Schmiester, Leonard, Yannik Schälte, Fabian Fröhlich, Jan Hasenauer, and Daniel Weindl. 2019. “Efficient parameterization of large-scale dynamic models based on relative measurements.” <em>Bioinformatics</em>, July. <a href="https://doi.org/10.1093/bioinformatics/btz581">https://doi.org/10.1093/bioinformatics/btz581</a>.</p>
</div>
<div id="ref-LinesPas2019">
<p>Terje Lines, Glenn, Łukasz Paszkowski, Leonard Schmiester, Daniel Weindl, Paul Stapor, and Jan Hasenauer. 2019. “Efficient Computation of Steady States in Large-Scale Ode Models of Biochemical Reaction Networks.” <em>IFAC-PapersOnLine</em> 52 (26): 32–37. <a href="https://doi.org/10.1016/j.ifacol.2019.12.232">https://doi.org/10.1016/j.ifacol.2019.12.232</a>.</p>
</div>
<div id="ref-VillaverdeRai2019">
<p>Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R. Banga. 2019. “A Comparison of Methods for Quantifying Prediction Uncertainty in Systems Biology.” <em>IFAC-PapersOnLine</em> 52 (26): 45–51. <a href="https://doi.org/10.1016/j.ifacol.2019.12.234">https://doi.org/10.1016/j.ifacol.2019.12.234</a>.</p>
</div>
<div id="ref-WangSta2019">
<p>Wang, Dantong, Paul Stapor, and Jan Hasenauer. 2019. “Dirac Mixture Distributions for the Approximation of Mixed Effects Models.” <em>IFAC-PapersOnLine</em> 52 (26): 200–206. <a href="https://doi.org/10.1016/j.ifacol.2019.12.258">https://doi.org/10.1016/j.ifacol.2019.12.258</a>.</p>
</div>
<div id="ref-Watanabe2019">
<p>Watanabe, Simon Berglund. 2019. “Identiﬁability of Parameters in Pbpk Models.” Master’s thesis, Chalmers University of Technology / Department of Mathematical Sciences. <a href="https://hdl.handle.net/20.500.12380/256855">https://hdl.handle.net/20.500.12380/256855</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2018</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-BallnusSch2018">
<p>Ballnus, Benjamin, Steffen Schaper, Fabian J Theis, and Jan Hasenauer. 2018. “Bayesian Parameter Estimation for Biochemical Reaction Networks Using Region-Based Adaptive Parallel Tempering.” <em>Bioinformatics</em> 34 (13): i494–i501. <a href="https://doi.org/10.1093/bioinformatics/bty229">https://doi.org/10.1093/bioinformatics/bty229</a>.</p>
</div>
<div id="ref-BastCal2018">
<p>Bast, Lisa, Filippo Calzolari, Michael Strasser, Jan Hasenauer, Fabian J. Theis, Jovica Ninkovic, and Carsten Marr. 2018. “Subtle Changes in Clonal Dynamics Underlie the Age-Related Decline in Neurogenesis.” <em>Cell Reports</em>. <a href="https://doi.org/10.1016/j.celrep.2018.11.088">https://doi.org/10.1016/j.celrep.2018.11.088</a>.</p>
</div>
<div id="ref-FroehlichKes2018">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6): 567–579.e6. <a href="https://doi.org/10.1016/j.cels.2018.10.013">https://doi.org/10.1016/j.cels.2018.10.013</a>.</p>
</div>
<div id="ref-FroehlichRei2018">
<p>Fröhlich, Fabian, Anita Reiser, Laura Fink, Daniel Woschée, Thomas Ligon, Fabian Joachim Theis, Joachim Oskar Rädler, and Jan Hasenauer. 2018. “Multi-Experiment Nonlinear Mixed Effect Modeling of Single-Cell Translation Kinetics After Transfection.” <em>Npj Systems Biology and Applications</em> 5 (1): 1. <a href="https://doi.org/10.1038/s41540-018-0079-7">https://doi.org/10.1038/s41540-018-0079-7</a>.</p>
</div>
<div id="ref-KaltenbacherPed2018">
<p>Kaltenbacher, Barbara, and Barbara Pedretscher. 2018. “Parameter Estimation in Sdes via the Fokker–Planck Equation: Likelihood Function and Adjoint Based Gradient Computation.” <em>Journal of Mathematical Analysis and Applications</em> 465 (2): 872–84. <a href="https://doi.org/10.1016/j.jmaa.2018.05.048">https://doi.org/10.1016/j.jmaa.2018.05.048</a>.</p>
</div>
<div id="ref-LoosKra2018">
<p>Loos, Carolin, Sabrina Krause, and Jan Hasenauer. 2018. “Hierarchical Optimization for the Efficient Parametrization of ODE Models.” <em>Bioinformatics</em> 34 (24): 4266–73. <a href="https://doi.org/10.1093/bioinformatics/bty514">https://doi.org/10.1093/bioinformatics/bty514</a>.</p>
</div>
<div id="ref-LoosMoe2018">
<p>Loos, Carolin, Katharina Moeller, Fabian Fröhlich, Tim Hucho, and Jan Hasenauer. 2018. “A Hierarchical, Data-Driven Approach to Modeling Single-Cell Populations Predicts Latent Causes of Cell-to-Cell Variability.” <em>Cell Systems</em> 6 (5): 593–603. <a href="https://doi.org/10.1016/j.cels.2018.04.008">https://doi.org/10.1016/j.cels.2018.04.008</a>.</p>
</div>
<div id="ref-PittGom2018">
<p>Pitt, Jake Alan, Lucian Gomoescu, Constantinos C. Pantelides, Benoît Chachuat, and Julio R. Banga. 2018. “Critical Assessment of Parameter Estimation Methods in Models of Biological Oscillators.” <em>IFAC-PapersOnLine</em> 51 (19): 72–75. <a href="https://doi.org/10.1016/j.ifacol.2018.09.040">https://doi.org/10.1016/j.ifacol.2018.09.040</a>.</p>
</div>
<div id="ref-SchaelteSta2018">
<p>Schälte, Y., P. Stapor, and J. Hasenauer. 2018. “Evaluation of Derivative-Free Optimizers for Parameter Estimation in Systems Biology.” <em>FAC-PapersOnLine</em> 51 (19): 98–101. <a href="https://doi.org/10.1016/j.ifacol.2018.09.025">https://doi.org/10.1016/j.ifacol.2018.09.025</a>.</p>
</div>
<div id="ref-StaporFro2018">
<p>Stapor, Paul, Fabian Fröhlich, and Jan Hasenauer. 2018. “Optimization and Profile Calculation of ODE Models Using Second Order Adjoint Sensitivity Analysis.” <em>Bioinformatics</em> 34 (13): i151–i159. <a href="https://doi.org/10.1093/bioinformatics/bty230">https://doi.org/10.1093/bioinformatics/bty230</a>.</p>
</div>
<div id="ref-VillaverdeFroe2018">
<p>Villaverde, Alejandro F, Fabian Fröhlich, Daniel Weindl, Jan Hasenauer, and Julio R Banga. 2018. “Benchmarking optimization methods for parameter estimation in large kinetic models.” <em>Bioinformatics</em> 35 (5): 830–38. <a href="https://doi.org/10.1093/bioinformatics/bty736">https://doi.org/10.1093/bioinformatics/bty736</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2017</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-BallnusHug2017">
<p>Ballnus, B., S. Hug, K. Hatz, L. Görlitz, J. Hasenauer, and F. J. Theis. 2017. “Comprehensive Benchmarking of Markov Chain Monte Carlo Methods for Dynamical Systems.” <em>BMC Syst. Biol.</em> 11 (63). <a href="https://doi.org/10.1186/s12918-017-0433-1">https://doi.org/10.1186/s12918-017-0433-1</a>.</p>
</div>
<div id="ref-FroehlichKal2017">
<p>Fröhlich, F., B. Kaltenbacher, F. J. Theis, and J. Hasenauer. 2017. “Scalable Parameter Estimation for Genome-Scale Biochemical Reaction Networks.” <em>PLoS Comput. Biol.</em> 13 (1): e1005331. <a href="https://doi.org/10.1371/journal.pcbi.1005331">https://doi.org/10.1371/journal.pcbi.1005331</a>.</p>
</div>
<div id="ref-FroehlichThe2016">
<p>Fröhlich, F., F. J. Theis, J. O. Rädler, and J. Hasenauer. 2017. “Parameter Estimation for Dynamical Systems with Discrete Events and Logical Operations.” <em>Bioinformatics</em> 33 (7): 1049–56. <a href="https://doi.org/10.1093/bioinformatics/btw764">https://doi.org/10.1093/bioinformatics/btw764</a>.</p>
</div>
<div id="ref-KazeroonianThe2017">
<p>Kazeroonian, A., F. J. Theis, and J. Hasenauer. 2017. “A Scalable Moment-Closure Approximation for Large-Scale Biochemical Reaction Networks.” <em>Bioinformatics</em> 33 (14): i293–i300. <a href="https://doi.org/10.1093/bioinformatics/btx249">https://doi.org/10.1093/bioinformatics/btx249</a>.</p>
</div>
<div id="ref-MaierLoo2017">
<p>Maier, C., C. Loos, and J. Hasenauer. 2017. “Robust Parameter Estimation for Dynamical Systems from Outlier-Corrupted Data.” <em>Bioinformatics</em> 33 (5): 718–25. <a href="https://doi.org/10.1093/bioinformatics/btw703">https://doi.org/10.1093/bioinformatics/btw703</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2016</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-BoigerHas2016">
<p>Boiger, R., J. Hasenauer, S. Hross, and B. Kaltenbacher. 2016. “Integration Based Profile Likelihood Calculation for PDE Constrained Parameter Estimation Problems.” <em>Inverse Prob.</em> 32 (12): 125009. <a href="https://doi.org/10.1088/0266-5611/32/12/125009">https://doi.org/10.1088/0266-5611/32/12/125009</a>.</p>
</div>
<div id="ref-FiedlerRae2016">
<p>Fiedler, A., S. Raeth, F. J. Theis, A. Hausser, and J. Hasenauer. 2016. “Tailored Parameter Optimization Methods for Ordinary Differential Equation Models with Steady-State Constraints.” <em>BMC Syst. Biol.</em> 10 (80). <a href="https://doi.org/10.1186/s12918-016-0319-7">https://doi.org/10.1186/s12918-016-0319-7</a>.</p>
</div>
<div id="ref-FroehlichTho2016">
<p>Fröhlich, F., P. Thomas, A. Kazeroonian, F. J. Theis, R. Grima, and J. Hasenauer. 2016. “Inference for Stochastic Chemical Kinetics Using Moment Equations and System Size Expansion.” <em>PLoS Comput. Biol.</em> 12 (7): e1005030. <a href="https://doi.org/10.1371/journal.pcbi.1005030">https://doi.org/10.1371/journal.pcbi.1005030</a>.</p>
</div>
<div id="ref-HrossFie2016">
<p>Hross, S., A. Fiedler, F. J. Theis, and J. Hasenauer. 2016. “Quantitative Comparison of Competing PDE Models for Pom1p Dynamics in Fission Yeast.” In <em>Proc. 6th IFAC Conf. Found. Syst. Biol. Eng.</em>, edited by R. Findeisen, E. Bullinger, E. Balsa-Canto, and K. Bernaerts, 49:264–69. 26. IFAC-PapersOnLine. <a href="https://doi.org/10.1016/j.ifacol.2016.12.136">https://doi.org/10.1016/j.ifacol.2016.12.136</a>.</p>
</div>
<div id="ref-KazeroonianFro2016">
<p>Kazeroonian, A., F. Fröhlich, A. Raue, F. J. Theis, and J. Hasenauer. 2016. “CERENA: Chemical REaction Network Analyzer – A Toolbox for the Simulation and Analysis of Stochastic Chemical Kinetics.” <em>PLoS ONE</em> 11 (1): e0146732. <a href="https://doi.org/10.1371/journal.pone.0146732">https://doi.org/10.1371/journal.pone.0146732</a>.</p>
</div>
<div id="ref-LoosFie2016">
<p>Loos, C., A. Fiedler, and J. Hasenauer. 2016. “Parameter Estimation for Reaction Rate Equation Constrained Mixture Models.” In <em>Proc. 13th Int. Conf. Comp. Meth. Syst. Biol.</em>, edited by E. Bartocci, P. Lio, and N. Paoletti, 186–200. Lecture Notes in Bioinformatics. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-45177-0">https://doi.org/10.1007/978-3-319-45177-0</a>.</p>
</div>
</div>
<h1 class="unnumbered" id="section">2015</h1>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-LoosMar2015">
<p>Loos, C., C. Marr, F. J. Theis, and J. Hasenauer. 2015. “Computational Methods in Systems Biology.” In, edited by O. Roux and J. Bourdon, 9308:52–63. Lecture Notes in Computer Science. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-23401-4_6">https://doi.org/10.1007/978-3-319-23401-4_6</a>.</p>
</div>
</div>
