# References

List of publications using AMICI. Total number is 37.

If you applied AMICI in your work and your publication is missing, please let us know via a new Github issue.

<h1 id="section" class="unnumbered">2020</h1>
<div id="refs" class="references">
<div id="ref-AlabertLoo2020">
<p>Alabert, Constance, Carolin Loos, Moritz Voelker-Albert, Simona Graziano, Ignasi Forné, Nazaret Reveron-Gomez, Lea Schuh, et al. 2020. “Domain Model Explains Propagation Dynamics and Stability of Histone H3k27 and H3k36 Methylation Landscapes.” <em>Cell Reports</em> 30 (4): 1223–1234.e8. <a href="https://doi.org/10.1016/j.celrep.2019.12.060">https://doi.org/10.1016/j.celrep.2019.12.060</a>.</p>
</div>
<div id="ref-SchmiesterWei2020">
<p>Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2020. “Parameterization of Mechanistic Models from Qualitative Data Using an Efficient Optimal Scaling Approach.” <em>Journal of Mathematical Biology</em>, July. <a href="https://doi.org/10.1007/s00285-020-01522-w">https://doi.org/10.1007/s00285-020-01522-w</a>.</p>
</div>
<div id="ref-Staedter2020.09.03.268276">
<p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2020. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.09.03.268276">https://doi.org/10.1101/2020.09.03.268276</a>.</p>
</div>
</div>
<h1 id="section" class="unnumbered">2019</h1>
<div id="refs" class="references">
<div id="ref-DharmarajanKal2019">
<p>Dharmarajan, Lekshmi, Hans-Michael Kaltenbach, Fabian Rudolf, and Joerg Stelling. 2019. “A Simple and Flexible Computational Framework for Inferring Sources of Heterogeneity from Single-Cell Dynamics.” <em>Cell Systems</em> 8 (1): 15–26.e11. <a href="https://doi.org/10.1016/j.cels.2018.12.007">https://doi.org/10.1016/j.cels.2018.12.007</a>.</p>
</div>
<div id="ref-FischerFie2017">
<p>Fischer, David S., Anna K. Fiedler, Eric Kernfeld, Ryan M. J. Genga, Aimée Bastidas-Ponce, Mostafa Bakhti, Heiko Lickert, Jan Hasenauer, Rene Maehr, and Fabian J. Theis. 2019. “Inferring Population Dynamics from Single-Cell Rna-Sequencing Time Series Data.” <em>Nature Biotechnology</em> 37: 461–68. <a href="https://doi.org/10.1038/s41587-019-0088-0">https://doi.org/10.1038/s41587-019-0088-0</a>.</p>
</div>
<div id="ref-GreggSar2019">
<p>Gregg, Robert W, Saumendra N Sarkar, and Jason E Shoemaker. 2019. “Mathematical Modeling of the cGAS Pathway Reveals Robustness of Dna Sensing to Trex1 Feedback.” <em>Journal of Theoretical Biology</em> 462 (February): 148–57. <a href="https://doi.org/10.1016/j.jtbi.2018.11.001">https://doi.org/10.1016/j.jtbi.2018.11.001</a>.</p>
</div>
<div id="ref-KapferSta2019">
<p>Kapfer, Eva-Maria, Paul Stapor, and Jan Hasenauer. 2019. “Challenges in the Calibration of Large-Scale Ordinary Differential Equation Models.” <em>IFAC-PapersOnLine</em> 52 (26): 58–64. <a href="https://doi.org/10.1016/j.ifacol.2019.12.236">https://doi.org/10.1016/j.ifacol.2019.12.236</a>.</p>
</div>
<div id="ref-NousiainenInt2019">
<p>Nousiainen, Kari, Jukka Intosalmi, and Harri Lähdesmäki. 2019. “A Mathematical Model for Enhancer Activation Kinetics During Cell Differentiation.” In <em>Algorithms for Computational Biology</em>, edited by Ian Holmes, Carlos Martı́n-Vide, and Miguel A. Vega-Rodrı́guez, 191–202. Cham: Springer International Publishing. <a href="https://doi.org/10.1007/978-3-030-18174-1_14">https://doi.org/10.1007/978-3-030-18174-1_14</a>.</p>
</div>
<div id="ref-PedretscherKal2019">
<p>Pedretscher, B., B. Kaltenbacher, and O. Pfeiler. 2019. “Parameter Identification and Uncertainty Quantification in Stochastic State Space Models and Its Application to Texture Analysis.” <em>Applied Numerical Mathematics</em> 146: 38–54. <a href="https://doi.org/10.1016/j.apnum.2019.06.020">https://doi.org/10.1016/j.apnum.2019.06.020</a>.</p>
</div>
<div id="ref-PittBan2019">
<p>Pitt, Jake Alan, and Julio R Banga. 2019. “Parameter Estimation in Models of Biological Oscillators: An Automated Regularised Estimation Approach.” <em>BMC Bioinformatics</em> 20 (1): 82. <a href="https://doi.org/10.1186/s12859-019-2630-y">https://doi.org/10.1186/s12859-019-2630-y</a>.</p>
</div>
<div id="ref-SchmiesterSch2019">
<p>Schmiester, Leonard, Yannik Schälte, Fabian Fröhlich, Jan Hasenauer, and Daniel Weindl. 2019. “Efficient parameterization of large-scale dynamic models based on relative measurements.” <em>Bioinformatics</em>, July. <a href="https://doi.org/10.1093/bioinformatics/btz581">https://doi.org/10.1093/bioinformatics/btz581</a>.</p>
</div>
<div id="ref-StaporSch2019">
<p>Stapor, Paul, Leonard Schmiester, Christoph Wierling, Bodo Lange, Daniel Weindl, and Jan Hasenauer. 2019. “Mini-Batch Optimization Enables Training of Ode Models on Large-Scale Datasets.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/859884">https://doi.org/10.1101/859884</a>.</p>
</div>
<div id="ref-LinesPas2019">
<p>Terje Lines, Glenn, Łukasz Paszkowski, Leonard Schmiester, Daniel Weindl, Paul Stapor, and Jan Hasenauer. 2019. “Efficient Computation of Steady States in Large-Scale Ode Models of Biochemical Reaction Networks.” <em>IFAC-PapersOnLine</em> 52 (26): 32–37. <a href="https://doi.org/10.1016/j.ifacol.2019.12.232">https://doi.org/10.1016/j.ifacol.2019.12.232</a>.</p>
</div>
<div id="ref-VillaverdeRai2019">
<p>Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R. Banga. 2019. “A Comparison of Methods for Quantifying Prediction Uncertainty in Systems Biology⁎⁎This Research Has Received Funding from the European Unions Horizon 2020 Research and Innovation Program Under Grant Agreement No 686282 (Canpathpro) and the German Ministry of Education and Research (Bmbf) Under the Grant Agreement No 01ZX1310B (Sys-Stomach) and No 01ZX1705A (Income).” <em>IFAC-PapersOnLine</em> 52 (26): 45–51. <a href="https://doi.org/10.1016/j.ifacol.2019.12.234">https://doi.org/10.1016/j.ifacol.2019.12.234</a>.</p>
</div>
<div id="ref-WangSta2019">
<p>Wang, Dantong, Paul Stapor, and Jan Hasenauer. 2019. “Dirac Mixture Distributions for the Approximation of Mixed Effects Models.” <em>IFAC-PapersOnLine</em> 52 (26): 200–206. <a href="https://doi.org/10.1016/j.ifacol.2019.12.258">https://doi.org/10.1016/j.ifacol.2019.12.258</a>.</p>
</div>
</div>
<h1 id="section" class="unnumbered">2018</h1>
<div id="refs" class="references">
<div id="ref-BallnusSch2018">
<p>Ballnus, Benjamin, Steffen Schaper, Fabian J Theis, and Jan Hasenauer. 2018. “Bayesian Parameter Estimation for Biochemical Reaction Networks Using Region-Based Adaptive Parallel Tempering.” <em>Bioinformatics</em> 34 (13): i494–i501. <a href="https://doi.org/10.1093/bioinformatics/bty229">https://doi.org/10.1093/bioinformatics/bty229</a>.</p>
</div>
<div id="ref-BastCal2018">
<p>Bast, Lisa, Filippo Calzolari, Michael Strasser, Jan Hasenauer, Fabian J. Theis, Jovica Ninkovic, and Carsten Marr. 2018. “Subtle Changes in Clonal Dynamics Underlie the Age-Related Decline in Neurogenesis.” <em>Cell Reports</em>. <a href="https://doi.org/10.1016/j.celrep.2018.11.088">https://doi.org/10.1016/j.celrep.2018.11.088</a>.</p>
</div>
<div id="ref-FroehlichKes2018">
<p>Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. “Efficient Parameter Estimation Enables the Prediction of Drug Response Using a Mechanistic Pan-Cancer Pathway Model.” <em>Cell Systems</em> 7 (6): 567–579.e6. <a href="https://doi.org/10.1016/j.cels.2018.10.013">https://doi.org/10.1016/j.cels.2018.10.013</a>.</p>
</div>
<div id="ref-FroehlichRei2018">
<p>Fröhlich, Fabian, Anita Reiser, Laura Fink, Daniel Woschée, Thomas Ligon, Fabian Joachim Theis, Joachim Oskar Rädler, and Jan Hasenauer. 2018. “Multi-Experiment Nonlinear Mixed Effect Modeling of Single-Cell Translation Kinetics After Transfection.” <em>Npj Systems Biology and Applications</em> 5 (1): 1. <a href="https://doi.org/10.1038/s41540-018-0079-7">https://doi.org/10.1038/s41540-018-0079-7</a>.</p>
</div>
<div id="ref-KaltenbacherPed2018">
<p>Kaltenbacher, Barbara, and Barbara Pedretscher. 2018. “Parameter Estimation in Sdes via the Fokker–Planck Equation: Likelihood Function and Adjoint Based Gradient Computation.” <em>Journal of Mathematical Analysis and Applications</em> 465 (2): 872–84. <a href="https://doi.org/10.1016/j.jmaa.2018.05.048">https://doi.org/10.1016/j.jmaa.2018.05.048</a>.</p>
</div>
<div id="ref-LoosKra2018">
<p>Loos, Carolin, Sabrina Krause, and Jan Hasenauer. 2018. “Hierarchical Optimization for the Efficient Parametrization of ODE Models.” <em>Bioinformatics</em> 34 (24): 4266–73. <a href="https://doi.org/10.1093/bioinformatics/bty514">https://doi.org/10.1093/bioinformatics/bty514</a>.</p>
</div>
<div id="ref-LoosMoe2018">
<p>Loos, Carolin, Katharina Moeller, Fabian Fröhlich, Tim Hucho, and Jan Hasenauer. 2018. “A Hierarchical, Data-Driven Approach to Modeling Single-Cell Populations Predicts Latent Causes of Cell-to-Cell Variability.” <em>Cell Systems</em> 6 (5): 593–603. <a href="https://doi.org/10.1016/j.cels.2018.04.008">https://doi.org/10.1016/j.cels.2018.04.008</a>.</p>
</div>
<div id="ref-SchaelteSta2018">
<p>Schälte, Y., P. Stapor, and J. Hasenauer. 2018. “Evaluation of Derivative-Free Optimizers for Parameter Estimation in Systems Biology.” <em>FAC-PapersOnLine</em> 51 (19): 98–101. <a href="https://doi.org/10.1016/j.ifacol.2018.09.025">https://doi.org/10.1016/j.ifacol.2018.09.025</a>.</p>
</div>
<div id="ref-StaporFro2018">
<p>Stapor, Paul, Fabian Fröhlich, and Jan Hasenauer. 2018. “Optimization and Profile Calculation of ODE Models Using Second Order Adjoint Sensitivity Analysis.” <em>Bioinformatics</em> 34 (13): i151–i159. <a href="https://doi.org/10.1093/bioinformatics/bty230">https://doi.org/10.1093/bioinformatics/bty230</a>.</p>
</div>
<div id="ref-VillaverdeFroe2018">
<p>Villaverde, Alejandro F, Fabian Fröhlich, Daniel Weindl, Jan Hasenauer, and Julio R Banga. 2018. “Benchmarking optimization methods for parameter estimation in large kinetic models.” <em>Bioinformatics</em> 35 (5): 830–38. <a href="https://doi.org/10.1093/bioinformatics/bty736">https://doi.org/10.1093/bioinformatics/bty736</a>.</p>
</div>
</div>
<h1 id="section" class="unnumbered">2017</h1>
<div id="refs" class="references">
<div id="ref-BallnusHug2017">
<p>Ballnus, B., S. Hug, K. Hatz, L. Görlitz, J. Hasenauer, and F. J. Theis. 2017. “Comprehensive Benchmarking of Markov Chain Monte Carlo Methods for Dynamical Systems.” <em>BMC Syst. Biol.</em> 11 (63). <a href="https://doi.org/10.1186/s12918-017-0433-1">https://doi.org/10.1186/s12918-017-0433-1</a>.</p>
</div>
<div id="ref-FroehlichKal2017">
<p>Fröhlich, F., B. Kaltenbacher, F. J. Theis, and J. Hasenauer. 2017. “Scalable Parameter Estimation for Genome-Scale Biochemical Reaction Networks.” <em>PLoS Comput. Biol.</em> 13 (1): e1005331. <a href="https://doi.org/10.1371/journal.pcbi.1005331">https://doi.org/10.1371/journal.pcbi.1005331</a>.</p>
</div>
<div id="ref-FroehlichThe2016">
<p>Fröhlich, F., F. J. Theis, J. O. Rädler, and J. Hasenauer. 2017. “Parameter Estimation for Dynamical Systems with Discrete Events and Logical Operations.” <em>Bioinformatics</em> 33 (7): 1049–56. <a href="https://doi.org/10.1093/bioinformatics/btw764">https://doi.org/10.1093/bioinformatics/btw764</a>.</p>
</div>
<div id="ref-KazeroonianThe2017">
<p>Kazeroonian, A., F. J. Theis, and J. Hasenauer. 2017. “A Scalable Moment-Closure Approximation for Large-Scale Biochemical Reaction Networks.” <em>Bioinformatics</em> 33 (14): i293–i300. <a href="https://doi.org/10.1093/bioinformatics/btx249">https://doi.org/10.1093/bioinformatics/btx249</a>.</p>
</div>
<div id="ref-MaierLoo2017">
<p>Maier, C., C. Loos, and J. Hasenauer. 2017. “Robust Parameter Estimation for Dynamical Systems from Outlier-Corrupted Data.” <em>Bioinformatics</em> 33 (5): 718–25. <a href="https://doi.org/10.1093/bioinformatics/btw703">https://doi.org/10.1093/bioinformatics/btw703</a>.</p>
</div>
</div>
<h1 id="section" class="unnumbered">2016</h1>
<div id="refs" class="references">
<div id="ref-BoigerHas2016">
<p>Boiger, R., J. Hasenauer, S. Hross, and B. Kaltenbacher. 2016. “Integration Based Profile Likelihood Calculation for PDE Constrained Parameter Estimation Problems.” <em>Inverse Prob.</em> 32 (12): 125009. <a href="https://doi.org/10.1088/0266-5611/32/12/125009">https://doi.org/10.1088/0266-5611/32/12/125009</a>.</p>
</div>
<div id="ref-FiedlerRae2016">
<p>Fiedler, A., S. Raeth, F. J. Theis, A. Hausser, and J. Hasenauer. 2016. “Tailored Parameter Optimization Methods for Ordinary Differential Equation Models with Steady-State Constraints.” <em>BMC Syst. Biol.</em> 10 (80). <a href="https://doi.org/10.1186/s12918-016-0319-7">https://doi.org/10.1186/s12918-016-0319-7</a>.</p>
</div>
<div id="ref-FroehlichTho2016">
<p>Fröhlich, F., P. Thomas, A. Kazeroonian, F. J. Theis, R. Grima, and J. Hasenauer. 2016. “Inference for Stochastic Chemical Kinetics Using Moment Equations and System Size Expansion.” <em>PLoS Comput. Biol.</em> 12 (7): e1005030. <a href="https://doi.org/10.1371/journal.pcbi.1005030">https://doi.org/10.1371/journal.pcbi.1005030</a>.</p>
</div>
<div id="ref-HrossFie2016">
<p>Hross, S., A. Fiedler, F. J. Theis, and J. Hasenauer. 2016. “Quantitative Comparison of Competing PDE Models for Pom1p Dynamics in Fission Yeast.” In <em>Proc. 6th IFAC Conf. Found. Syst. Biol. Eng.</em>, edited by R. Findeisen, E. Bullinger, E. Balsa-Canto, and K. Bernaerts, 49:264–69. 26. IFAC-PapersOnLine. <a href="https://doi.org/10.1016/j.ifacol.2016.12.136">https://doi.org/10.1016/j.ifacol.2016.12.136</a>.</p>
</div>
<div id="ref-KazeroonianFro2016">
<p>Kazeroonian, A., F. Fröhlich, A. Raue, F. J. Theis, and J. Hasenauer. 2016. “CERENA: Chemical REaction Network Analyzer – A Toolbox for the Simulation and Analysis of Stochastic Chemical Kinetics.” <em>PLoS ONE</em> 11 (1): e0146732. <a href="https://doi.org/10.1371/journal.pone.0146732">https://doi.org/10.1371/journal.pone.0146732</a>.</p>
</div>
<div id="ref-LoosFie2016">
<p>Loos, C., A. Fiedler, and J. Hasenauer. 2016. “Parameter Estimation for Reaction Rate Equation Constrained Mixture Models.” In <em>Proc. 13th Int. Conf. Comp. Meth. Syst. Biol.</em>, edited by E. Bartocci, P. Lio, and N. Paoletti, 186–200. Lecture Notes in Bioinformatics. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-45177-0">https://doi.org/10.1007/978-3-319-45177-0</a>.</p>
</div>
</div>
<h1 id="section" class="unnumbered">2015</h1>
<div id="refs" class="references">
<div id="ref-LoosMar2015">
<p>Loos, C., C. Marr, F. J. Theis, and J. Hasenauer. 2015. “Computational Methods in Systems Biology.” In, edited by O. Roux and J. Bourdon, 9308:52–63. Lecture Notes in Computer Science. Springer International Publishing. <a href="https://doi.org/10.1007/978-3-319-23401-4_6">https://doi.org/10.1007/978-3-319-23401-4_6</a>.</p>
</div>
</div>
