# References

List of publications using AMICI. Total number is 83.

If you applied AMICI in your work and your publication is missing, please let us know via a new GitHub issue.


<style>
.csl-entry {
    padding: 5px
}
</style>

<h1 class="unnumbered" id="section">2024</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-LangPen2024" class="csl-entry" role="listitem">
Lang, Paul F., David R. Penas, Julio R. Banga, Daniel Weindl, and Bela
Novak. 2024. <span>“Reusable Rule-Based Cell Cycle Model Explains
Compartment-Resolved Dynamics of 16 Observables in RPE-1 Cells.”</span>
<em>PLOS Computational Biology</em> 20 (1): 1–24. <a
href="https://doi.org/10.1371/journal.pcbi.1011151">https://doi.org/10.1371/journal.pcbi.1011151</a>.
</div>
<div id="ref-SluijsZho2024" class="csl-entry" role="listitem">
Sluijs, Bob van, Tao Zhou, Britta Helwig, Mathieu G. Baltussen, Frank H.
T. Nelissen, Hans A. Heus, and Wilhelm T. S. Huck. 2024.
<span>“Iterative Design of Training Data to Control Intricate Enzymatic
Reaction Networks.”</span> <em>Nature Communications</em> 15 (1). <a
href="https://doi.org/10.1038/s41467-024-45886-9">https://doi.org/10.1038/s41467-024-45886-9</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2023</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-BuckBas2023" class="csl-entry" role="listitem">
Buck, Michèle C., Lisa Bast, Judith S. Hecker, Jennifer Rivière, Maja
Rothenberg-Thurley, Luisa Vogel, Dantong Wang, et al. 2023.
<span>“Progressive Disruption of Hematopoietic Architecture from Clonal
Hematopoiesis to <span>MDS</span>.”</span> <em>iScience</em>, 107328. <a
href="https://doi.org/10.1016/j.isci.2023.107328">https://doi.org/10.1016/j.isci.2023.107328</a>.
</div>
<div id="ref-ContentoCas2023" class="csl-entry" role="listitem">
Contento, Lorenzo, Noemi Castelletti, Elba Raimúndez, Ronan Le Gleut,
Yannik Schälte, Paul Stapor, Ludwig Christian Hinske, et al. 2023.
<span>“Integrative Modelling of Reported Case Numbers and Seroprevalence
Reveals Time-Dependent Test Efficiency and Infectious Contacts.”</span>
<em>Epidemics</em> 43: 100681. <a
href="https://doi.org/10.1016/j.epidem.2023.100681">https://doi.org/10.1016/j.epidem.2023.100681</a>.
</div>
<div id="ref-ContentoSta2023" class="csl-entry" role="listitem">
Contento, Lorenzo, Paul Stapor, Daniel Weindl, and Jan Hasenauer. 2023.
<span>“A More Expressive Spline Representation for <span>SBML</span>
Models Improves Code Generation Performance in
<span>AMICI</span>.”</span> <em>bioRxiv</em>. <a
href="https://doi.org/10.1101/2023.06.29.547120">https://doi.org/10.1101/2023.06.29.547120</a>.
</div>
<div id="ref-Froehlich2023" class="csl-entry" role="listitem">
Fröhlich, Fabian. 2023. <span>“A Practical Guide for the Efficient
Formulation and Calibration of Large, Energy- and Rule-Based Models of
Cellular Signal Transduction.”</span> In <em>Computational Modeling of
Signaling Networks</em>, edited by Lan K. Nguyen, 59–86. New York, NY:
Springer US. <a
href="https://doi.org/10.1007/978-1-0716-3008-2_3">https://doi.org/10.1007/978-1-0716-3008-2_3</a>.
</div>
<div id="ref-FroehlichGer2023" class="csl-entry" role="listitem">
Fröhlich, Fabian, Luca Gerosa, Jeremy Muhlich, and Peter K Sorger. 2023.
<span>“Mechanistic Model of MAPK Signaling Reveals How Allostery and
Rewiring Contribute to Drug Resistance.”</span> <em>Molecular Systems
Biology</em> 19 (2): e10988. <a
href="https://doi.org/10.15252/msb.202210988">https://doi.org/10.15252/msb.202210988</a>.
</div>
<div id="ref-HasenauerMer2023" class="csl-entry" role="listitem">
Hasenauer, Jan, Simon Merkt, Solomon Ali, Esayas Gudina, Wondimagegn
Adissu, Maximilian Münchhoff, Alexander Graf, et al. 2023.
<span>“Long-Term Monitoring of SARS-CoV-2 Seroprevalence and Variants in
Ethiopia Provides Prediction for Immunity and Cross-Immunity.”</span> <a
href="https://doi.org/10.21203/rs.3.rs-3307821/v1">https://doi.org/10.21203/rs.3.rs-3307821/v1</a>.
</div>
<div id="ref-HuckBal2023" class="csl-entry" role="listitem">
Huck, Wilhelm, Mathieu Baltussen, Thijs de Jong, Quentin Duez, and
William Robinson. 2023. <span>“Chemical Reservoir Computation in a
Self-Organizing Reaction Network.”</span> Research Square Platform LLC.
<a
href="https://doi.org/10.21203/rs.3.rs-3487081/v1">https://doi.org/10.21203/rs.3.rs-3487081/v1</a>.
</div>
<div id="ref-LakrisenkoSta2023" class="csl-entry" role="listitem">
Lakrisenko, Polina, Paul Stapor, Stephan Grein, Łukasz Paszkowski, Dilan
Pathirana, Fabian Fröhlich, Glenn Terje Lines, Daniel Weindl, and Jan
Hasenauer. 2023. <span>“Efficient Computation of Adjoint Sensitivities
at Steady-State in ODE Models of Biochemical Reaction Networks.”</span>
<em>PLOS Computational Biology</em> 19 (1): 1–19. <a
href="https://doi.org/10.1371/journal.pcbi.1010783">https://doi.org/10.1371/journal.pcbi.1010783</a>.
</div>
<div id="ref-Mendes2023" class="csl-entry" role="listitem">
Mendes, Pedro. 2023. <span>“Reproducibility and FAIR Principles: The
Case of a Segment Polarity Network Model.”</span> <em>Frontiers in Cell
and Developmental Biology</em> 11. <a
href="https://doi.org/10.3389/fcell.2023.1201673">https://doi.org/10.3389/fcell.2023.1201673</a>.
</div>
<div id="ref-MishraWan2023" class="csl-entry" role="listitem">
Mishra, Shekhar, Ziyu Wang, Michael J. Volk, and Huimin Zhao. 2023.
<span>“Design and Application of a Kinetic Model of Lipid Metabolism in
Saccharomyces Cerevisiae.”</span> <em>Metabolic Engineering</em> 75:
12–18. <a
href="https://doi.org/10.1016/j.ymben.2022.11.003">https://doi.org/10.1016/j.ymben.2022.11.003</a>.
</div>
<div id="ref-RaimundezFed2023" class="csl-entry" role="listitem">
Raimúndez, Elba, Michael Fedders, and Jan Hasenauer. 2023.
<span>“Posterior Marginalization Accelerates Bayesian Inference for
Dynamical Models of Biological Processes.”</span>
<em><span>iScience</span></em>, September, 108083. <a
href="https://doi.org/10.1016/j.isci.2023.108083">https://doi.org/10.1016/j.isci.2023.108083</a>.
</div>
<div id="ref-TunedalVio2023" class="csl-entry" role="listitem">
Tunedal, Kajsa, Federica Viola, Belén Casas Garcia, Ann Bolger, Fredrik
H. Nyström, Carl Johan Östgren, Jan Engvall, et al. 2023.
<span>“Haemodynamic Effects of Hypertension and Type 2 Diabetes:
Insights from a <span>4D</span> Flow <span
class="nocase">MRI-based</span> Personalized Cardiovascular Mathematical
Model.”</span> <em>The Journal of Physiology</em> n/a (n/a). <a
href="https://doi.org/10.1113/JP284652">https://doi.org/10.1113/JP284652</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2022</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-AlbadryHoe2022" class="csl-entry" role="listitem">
Albadry, Mohamed, Sebastian Höpfl, Nadia Ehteshamzad, Matthias König,
Michael Böttcher, Jasna Neumann, Amelie Lupp, et al. 2022.
<span>“Periportal Steatosis in Mice Affects Distinct Parameters of
Pericentral Drug Metabolism.”</span> <em>Scientific Reports</em> 12 (1):
21825. <a
href="https://doi.org/10.1038/s41598-022-26483-6">https://doi.org/10.1038/s41598-022-26483-6</a>.
</div>
<div id="ref-ErdemMut2022" class="csl-entry" role="listitem">
Erdem, Cemal, Arnab Mutsuddy, Ethan M. Bensman, William B. Dodd, Michael
M. Saint-Antoine, Mehdi Bouhaddou, Robert C. Blake, et al. 2022.
<span>“A Scalable, Open-Source Implementation of a Large-Scale
Mechanistic Model for Single Cell Proliferation and Death
Signaling.”</span> <em>Nature Communications</em> 13 (1): 3555. <a
href="https://doi.org/10.1038/s41467-022-31138-1">https://doi.org/10.1038/s41467-022-31138-1</a>.
</div>
<div id="ref-FroehlichSor2022" class="csl-entry" role="listitem">
Fröhlich, Fabian, and Peter K. Sorger. 2022. <span>“Fides: Reliable
Trust-Region Optimization for Parameter Estimation of Ordinary
Differential Equation Models.”</span> <em>PLOS Computational
Biology</em> 18 (7): 1–28. <a
href="https://doi.org/10.1371/journal.pcbi.1010322">https://doi.org/10.1371/journal.pcbi.1010322</a>.
</div>
<div id="ref-MassonisVil2022" class="csl-entry" role="listitem">
Massonis, Gemma, Alejandro F Villaverde, and Julio R Banga. 2022.
<span>“<span class="nocase">Improving dynamic predictions with ensembles
of observable models</span>.”</span> <em>Bioinformatics</em>, November.
<a
href="https://doi.org/10.1093/bioinformatics/btac755">https://doi.org/10.1093/bioinformatics/btac755</a>.
</div>
<div id="ref-MeyerSoe2022" class="csl-entry" role="listitem">
Meyer, Kristian, Mikkel Søes Ibsen, Lisa Vetter-Joss, Ernst Broberg
Hansen, and Jens Abildskov. 2022. <span>“Industrial Ion-Exchange
Chromatography Development Using Discontinuous Galerkin Methods Coupled
with Forward Sensitivity Analysis.”</span> <em>Journal of Chromatography
A</em>, 463741. <a
href="https://doi.org/10.1016/j.chroma.2022.463741">https://doi.org/10.1016/j.chroma.2022.463741</a>.
</div>
<div id="ref-SchmuckerFar2022" class="csl-entry" role="listitem">
Schmucker, Robin, Gabriele Farina, James Faeder, Fabian Fröhlich, Ali
Sinan Saglam, and Tuomas Sandholm. 2022. <span>“Combination Treatment
Optimization Using a Pan-Cancer Pathway Model.”</span> <em>PLOS
Computational Biology</em> 17 (12): 1–22. <a
href="https://doi.org/10.1371/journal.pcbi.1009689">https://doi.org/10.1371/journal.pcbi.1009689</a>.
</div>
<div id="ref-SluijsMaa2022" class="csl-entry" role="listitem">
Sluijs, Bob van, Roel J. M. Maas, Ardjan J. van der Linden, Tom F. A. de
Greef, and Wilhelm T. S. Huck. 2022. <span>“A Microfluidic Optimal
Experimental Design Platform for Forward Design of Cell-Free Genetic
Networks.”</span> <em>Nature Communications</em> 13 (1): 3626. <a
href="https://doi.org/10.1038/s41467-022-31306-3">https://doi.org/10.1038/s41467-022-31306-3</a>.
</div>
<div id="ref-StaporSch2022" class="csl-entry" role="listitem">
Stapor, Paul, Leonard Schmiester, Christoph Wierling, Simon Merkt, Dilan
Pathirana, Bodo M. H. Lange, Daniel Weindl, and Jan Hasenauer. 2022.
<span>“<span class="nocase">Mini-batch optimization enables training of
ODE models on large-scale datasets</span>.”</span> <em>Nature
Communications</em> 13 (1): 34. <a
href="https://doi.org/10.1038/s41467-021-27374-6">https://doi.org/10.1038/s41467-021-27374-6</a>.
</div>
<div id="ref-SundqvistSte2022" class="csl-entry" role="listitem">
Sundqvist, Nicolas, Sebastian Sten, Peter Thompson, Benjamin Jan
Andersson, Maria Engström, and Gunnar Cedersund. 2022.
<span>“Mechanistic Model for Human Brain Metabolism and Its Connection
to the Neurovascular Coupling.”</span> <em>PLOS Computational
Biology</em> 18 (12): 1–24. <a
href="https://doi.org/10.1371/journal.pcbi.1010798">https://doi.org/10.1371/journal.pcbi.1010798</a>.
</div>
<div id="ref-VillaverdeRai2022" class="csl-entry" role="listitem">
Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R.
Banga. 2022. <span>“Assessment of Prediction Uncertainty Quantification
Methods in Systems Biology.”</span> <em>IEEE/ACM Transactions on
Computational Biology and Bioinformatics</em>, 1–12. <a
href="https://doi.org/10.1109/TCBB.2022.3213914">https://doi.org/10.1109/TCBB.2022.3213914</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2021</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-AdlungSta2021" class="csl-entry" role="listitem">
Adlung, Lorenz, Paul Stapor, Christian Tönsing, Leonard Schmiester,
Luisa E. Schwarzmüller, Lena Postawa, Dantong Wang, et al. 2021.
<span>“Cell-to-Cell Variability in JAK2/STAT5 Pathway Components and
Cytoplasmic Volumes Defines Survival Threshold in Erythroid Progenitor
Cells.”</span> <em>Cell Reports</em> 36 (6): 109507. <a
href="https://doi.org/10.1016/j.celrep.2021.109507">https://doi.org/10.1016/j.celrep.2021.109507</a>.
</div>
<div id="ref-BastBuc2021" class="csl-entry" role="listitem">
Bast, Lisa, Michèle C. Buck, Judith S. Hecker, Robert A. J. Oostendorp,
Katharina S. Götze, and Carsten Marr. 2021. <span>“Computational
Modeling of Stem and Progenitor Cell Kinetics Identifies Plausible
Hematopoietic Lineage Hierarchies.”</span> <em>iScience</em> 24 (2):
102120. <a
href="https://doi.org/10.1016/j.isci.2021.102120">https://doi.org/10.1016/j.isci.2021.102120</a>.
</div>
<div id="ref-Gaspari2021" class="csl-entry" role="listitem">
Gaspari, Erika. 2021. <span>“Model-Driven Design of Mycoplasma as a
Vaccine Chassis.”</span> PhD thesis, Wageningen: Wageningen University.
<a
href="https://doi.org/10.18174/539593">https://doi.org/10.18174/539593</a>.
</div>
<div id="ref-GudinaAli2021" class="csl-entry" role="listitem">
Gudina, Esayas Kebede, Solomon Ali, Eyob Girma, Addisu Gize,
Birhanemeskel Tegene, Gadissa Bedada Hundie, Wondewosen Tsegaye Sime, et
al. 2021. <span>“<span class="nocase">Seroepidemiology and model-based
prediction of SARS-CoV-2 in Ethiopia: longitudinal cohort study among
front-line hospital workers and communities</span>.”</span> <em>The
Lancet Global Health</em> 9 (11): e1517–27. <a
href="https://doi.org/10.1016/S2214-109X(21)00386-7">https://doi.org/10.1016/S2214-109X(21)00386-7</a>.
</div>
<div id="ref-Maier2021" class="csl-entry" role="listitem">
Maier, Corinna. 2021. <span>“Bayesian Data Assimilation and
Reinforcement Learning for Model-Informed Precision Dosing in
Oncology.”</span> Doctoralthesis, Universit<span>ä</span>t Potsdam. <a
href="https://doi.org/10.25932/publishup-51587">https://doi.org/10.25932/publishup-51587</a>.
</div>
<div id="ref-RaimundezDud2021" class="csl-entry" role="listitem">
Raimúndez, Elba, Erika Dudkin, Jakob Vanhoefer, Emad Alamoudi, Simon
Merkt, Lara Fuhrmann, Fan Bai, and Jan Hasenauer. 2021. <span>“COVID-19
Outbreak in Wuhan Demonstrates the Limitations of Publicly Available
Case Numbers for Epidemiological Modeling.”</span> <em>Epidemics</em>
34: 100439. <a
href="https://doi.org/10.1016/j.epidem.2021.100439">https://doi.org/10.1016/j.epidem.2021.100439</a>.
</div>
<div id="ref-Schmiester2021.02.06.430039" class="csl-entry"
role="listitem">
Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2021.
<span>“Efficient Gradient-Based Parameter Estimation for Dynamic Models
Using Qualitative Data.”</span> <em>bioRxiv</em>. <a
href="https://doi.org/10.1101/2021.02.06.430039">https://doi.org/10.1101/2021.02.06.430039</a>.
</div>
<div id="ref-StaedterSch2021" class="csl-entry" role="listitem">
Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and
Paul L. Stapor. 2021. <span>“Benchmarking of Numerical Integration
Methods for ODE Models of Biological Systems.”</span> <em>Scientific
Reports</em> 11 (1): 2696. <a
href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.
</div>
<div id="ref-StenPod2021" class="csl-entry" role="listitem">
Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder,
Maria Engström, and Gunnar Cedersund. 2021. <span>“A Multi-Data Based
Quantitative Model for the Neurovascular Coupling in the Brain.”</span>
<em>bioRxiv</em>. <a
href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.
</div>
<div id="ref-TomasoniPar2021" class="csl-entry" role="listitem">
Tomasoni, Danilo, Alessio Paris, Stefano Giampiccolo, Federico Reali,
Giulia Simoni, Luca Marchetti, Chanchala Kaddi, et al. 2021.
<span>“<span>QSPcc</span> Reduces Bottlenecks in Computational Model
Simulations.”</span> <em>Communications Biology</em> 4 (1): 1022. <a
href="https://doi.org/10.1038/s42003-021-02553-9">https://doi.org/10.1038/s42003-021-02553-9</a>.
</div>
<div id="ref-vanRosmalenSmi2021" class="csl-entry" role="listitem">
van Rosmalen, R. P., R. W. Smith, V. A. P. Martins dos Santos, C. Fleck,
and M. Suarez-Diez. 2021. <span>“Model Reduction of Genome-Scale
Metabolic Models as a Basis for Targeted Kinetic Models.”</span>
<em>Metabolic Engineering</em> 64: 74–84. <a
href="https://doi.org/10.1016/j.ymben.2021.01.008">https://doi.org/10.1016/j.ymben.2021.01.008</a>.
</div>
<div id="ref-VanhoeferMat2021" class="csl-entry" role="listitem">
Vanhoefer, Jakob, Marta R. A. Matos, Dilan Pathirana, Yannik Schälte,
and Jan Hasenauer. 2021. <span>“Yaml2sbml: Human-Readable and -Writable
Specification of <span>ODE</span> Models and Their Conversion to
<span>SBML</span>.”</span> <em>Journal of Open Source Software</em> 6
(61): 3215. <a
href="https://doi.org/10.21105/joss.03215">https://doi.org/10.21105/joss.03215</a>.
</div>
<div id="ref-VillaverdePat2021" class="csl-entry" role="listitem">
Villaverde, Alejandro F, Dilan Pathirana, Fabian Fröhlich, Jan
Hasenauer, and Julio R Banga. 2021. <span>“<span class="nocase">A
protocol for dynamic model calibration</span>.”</span> <em>Briefings in
Bioinformatics</em>, October. <a
href="https://doi.org/10.1093/bib/bbab387">https://doi.org/10.1093/bib/bbab387</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2020</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-AlabertLoo2020" class="csl-entry" role="listitem">
Alabert, Constance, Carolin Loos, Moritz Voelker-Albert, Simona
Graziano, Ignasi Forné, Nazaret Reveron-Gomez, Lea Schuh, et al. 2020.
<span>“Domain Model Explains Propagation Dynamics and Stability of
Histone H3K27 and H3K36 Methylation Landscapes.”</span> <em>Cell
Reports</em> 30 (January): 1223–1234.e8. <a
href="https://doi.org/10.1016/j.celrep.2019.12.060">https://doi.org/10.1016/j.celrep.2019.12.060</a>.
</div>
<div id="ref-ErdemBen2020" class="csl-entry" role="listitem">
Erdem, Cemal, Ethan M. Bensman, Arnab Mutsuddy, Michael M.
Saint-Antoine, Mehdi Bouhaddou, Robert C. Blake, Will Dodd, et al. 2020.
<span>“A Simple and Efficient Pipeline for Construction, Merging,
Expansion, and Simulation of Large-Scale, Single-Cell Mechanistic
Models.”</span> <em>bioRxiv</em>. <a
href="https://doi.org/10.1101/2020.11.09.373407">https://doi.org/10.1101/2020.11.09.373407</a>.
</div>
<div id="ref-GerosaChi2020" class="csl-entry" role="listitem">
Gerosa, Luca, Christopher Chidley, Fabian Fröhlich, Gabriela Sanchez,
Sang Kyun Lim, Jeremy Muhlich, Jia-Yun Chen, et al. 2020.
<span>“Receptor-Driven ERK Pulses Reconfigure MAPK Signaling and Enable
Persistence of Drug-Adapted BRAF-Mutant Melanoma Cells.”</span> <em>Cell
Systems</em>. <a
href="https://doi.org/10.1016/j.cels.2020.10.002">https://doi.org/10.1016/j.cels.2020.10.002</a>.
</div>
<div id="ref-Kuritz2020.03.30.015909" class="csl-entry" role="listitem">
Kuritz, Karsten, Alain R Bonny, João Pedro Fonseca, and Frank Allgöwer.
2020. <span>“PDE-Constrained Optimization for Estimating Population
Dynamics over Cell Cycle from Static Single Cell Measurements.”</span>
<em>bioRxiv</em>. <a
href="https://doi.org/10.1101/2020.03.30.015909">https://doi.org/10.1101/2020.03.30.015909</a>.
</div>
<div id="ref-MaierHar2020" class="csl-entry" role="listitem">
Maier, Corinna, Niklas Hartung, Charlotte Kloft, Wilhelm Huisinga, and
Jana de Wiljes. 2020. <span>“Reinforcement Learning and Bayesian Data
Assimilation for Model-Informed Precision Dosing in Oncology.”</span> <a
href="https://arxiv.org/abs/2006.01061">https://arxiv.org/abs/2006.01061</a>.
</div>
<div id="ref-SchaelteHas2020" class="csl-entry" role="listitem">
Schälte, Yannik, and Jan Hasenauer. 2020. <span>“<span
class="nocase">Efficient exact inference for dynamical systems with
noisy measurements using sequential approximate Bayesian
computation</span>.”</span> <em>Bioinformatics</em> 36 (Supplement_1):
i551–59. <a
href="https://doi.org/10.1093/bioinformatics/btaa397">https://doi.org/10.1093/bioinformatics/btaa397</a>.
</div>
<div id="ref-SchmiesterWei2020" class="csl-entry" role="listitem">
Schmiester, Leonard, Daniel Weindl, and Jan Hasenauer. 2020.
<span>“Parameterization of Mechanistic Models from Qualitative Data
Using an Efficient Optimal Scaling Approach.”</span> <em>Journal of
Mathematical Biology</em>, July. <a
href="https://doi.org/10.1007/s00285-020-01522-w">https://doi.org/10.1007/s00285-020-01522-w</a>.
</div>
<div id="ref-SchuhLoo2020" class="csl-entry" role="listitem">
Schuh, Lea, Carolin Loos, Daniil Pokrovsky, Axel Imhof, Ralph A. W.
Rupp, and Carsten Marr. 2020. <span>“H4K20 Methylation Is Differently
Regulated by Dilution and Demethylation in Proliferating and
Cell-Cycle-Arrested Xenopus Embryos.”</span> <em>Cell Systems</em> 11
(6): 653–662.e8. <a
href="https://doi.org/10.1016/j.cels.2020.11.003">https://doi.org/10.1016/j.cels.2020.11.003</a>.
</div>
<div id="ref-Sten2020" class="csl-entry" role="listitem">
Sten, Sebastian. 2020. <span>“Mathematical Modeling of Neurovascular
Coupling.”</span> Linköping University Medical Dissertations. PhD
thesis, Linköping UniversityLinköping UniversityLinköping University,
Division of Diagnostics; Specialist Medicine, Faculty of Medicine;
Health Sciences, Center for Medical Image Science; Visualization (CMIV);
Linköping University, Division of Diagnostics; Specialist Medicine. <a
href="https://doi.org/10.3384/diss.diva-167806">https://doi.org/10.3384/diss.diva-167806</a>.
</div>
<div id="ref-StenEli2020" class="csl-entry" role="listitem">
Sten, Sebastian, Fredrik Elinder, Gunnar Cedersund, and Maria Engström.
2020. <span>“A Quantitative Analysis of Cell-Specific Contributions and
the Role of Anesthetics to the Neurovascular Coupling.”</span>
<em>NeuroImage</em> 215: 116827. <a
href="https://doi.org/10.1016/j.neuroimage.2020.116827">https://doi.org/10.1016/j.neuroimage.2020.116827</a>.
</div>
<div id="ref-TsipaPit2020" class="csl-entry" role="listitem">
Tsipa, Argyro, Jake Alan Pitt, Julio R. Banga, and Athanasios
Mantalaris. 2020. <span>“A Dual-Parameter Identification Approach for
Data-Based Predictive Modeling of Hybrid Gene Regulatory Network-Growth
Kinetics in Pseudomonas Putida Mt-2.”</span> <em>Bioprocess and
Biosystems Engineering</em> 43 (9): 1671–88. <a
href="https://doi.org/10.1007/s00449-020-02360-2">https://doi.org/10.1007/s00449-020-02360-2</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2019</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-DharmarajanKal2019" class="csl-entry" role="listitem">
Dharmarajan, Lekshmi, Hans-Michael Kaltenbach, Fabian Rudolf, and Joerg
Stelling. 2019. <span>“A Simple and Flexible Computational Framework for
Inferring Sources of Heterogeneity from Single-Cell Dynamics.”</span>
<em>Cell Systems</em> 8 (1): 15–26.e11. <a
href="https://doi.org/10.1016/j.cels.2018.12.007">https://doi.org/10.1016/j.cels.2018.12.007</a>.
</div>
<div id="ref-FischerFie2017" class="csl-entry" role="listitem">
Fischer, David S., Anna K. Fiedler, Eric Kernfeld, Ryan M. J. Genga,
Aimée Bastidas-Ponce, Mostafa Bakhti, Heiko Lickert, Jan Hasenauer, Rene
Maehr, and Fabian J. Theis. 2019. <span>“Inferring Population Dynamics
from Single-Cell RNA-Sequencing Time Series Data.”</span> <em>Nature
Biotechnology</em> 37: 461–68. <a
href="https://doi.org/10.1038/s41587-019-0088-0">https://doi.org/10.1038/s41587-019-0088-0</a>.
</div>
<div id="ref-GreggSar2019" class="csl-entry" role="listitem">
Gregg, Robert W, Saumendra N Sarkar, and Jason E Shoemaker. 2019.
<span>“Mathematical Modeling of the cGAS Pathway Reveals Robustness of
DNA Sensing to TREX1 Feedback.”</span> <em>Journal of Theoretical
Biology</em> 462 (February): 148–57. <a
href="https://doi.org/10.1016/j.jtbi.2018.11.001">https://doi.org/10.1016/j.jtbi.2018.11.001</a>.
</div>
<div id="ref-KapferSta2019" class="csl-entry" role="listitem">
Kapfer, Eva-Maria, Paul Stapor, and Jan Hasenauer. 2019.
<span>“Challenges in the Calibration of Large-Scale Ordinary
Differential Equation Models.”</span> <em>IFAC-PapersOnLine</em> 52
(26): 58–64. <a
href="https://doi.org/10.1016/j.ifacol.2019.12.236">https://doi.org/10.1016/j.ifacol.2019.12.236</a>.
</div>
<div id="ref-NousiainenInt2019" class="csl-entry" role="listitem">
Nousiainen, Kari, Jukka Intosalmi, and Harri Lähdesmäki. 2019. <span>“A
Mathematical Model for Enhancer Activation Kinetics During Cell
Differentiation.”</span> In <em>Algorithms for Computational
Biology</em>, edited by Ian Holmes, Carlos Martı́n-Vide, and Miguel A.
Vega-Rodrı́guez, 191–202. Cham: Springer International Publishing. <a
href="https://doi.org/10.1007/978-3-030-18174-1_14">https://doi.org/10.1007/978-3-030-18174-1_14</a>.
</div>
<div id="ref-PedretscherKal2019" class="csl-entry" role="listitem">
Pedretscher, B., B. Kaltenbacher, and O. Pfeiler. 2019. <span>“Parameter
Identification and Uncertainty Quantification in Stochastic State Space
Models and Its Application to Texture Analysis.”</span> <em>Applied
Numerical Mathematics</em> 146: 38–54. <a
href="https://doi.org/10.1016/j.apnum.2019.06.020">https://doi.org/10.1016/j.apnum.2019.06.020</a>.
</div>
<div id="ref-PittBan2019" class="csl-entry" role="listitem">
Pitt, Jake Alan, and Julio R Banga. 2019. <span>“Parameter Estimation in
Models of Biological Oscillators: An Automated Regularised Estimation
Approach.”</span> <em>BMC Bioinformatics</em> 20 (February): 82. <a
href="https://doi.org/10.1186/s12859-019-2630-y">https://doi.org/10.1186/s12859-019-2630-y</a>.
</div>
<div id="ref-SchmiesterSch2019" class="csl-entry" role="listitem">
Schmiester, Leonard, Yannik Schälte, Fabian Fröhlich, Jan Hasenauer, and
Daniel Weindl. 2019. <span>“<span class="nocase">Efficient
parameterization of large-scale dynamic models based on relative
measurements</span>.”</span> <em>Bioinformatics</em>, July. <a
href="https://doi.org/10.1093/bioinformatics/btz581">https://doi.org/10.1093/bioinformatics/btz581</a>.
</div>
<div id="ref-LinesPas2019" class="csl-entry" role="listitem">
Terje Lines, Glenn, Łukasz Paszkowski, Leonard Schmiester, Daniel
Weindl, Paul Stapor, and Jan Hasenauer. 2019. <span>“Efficient
Computation of Steady States in Large-Scale ODE Models of Biochemical
Reaction Networks.”</span> <em>IFAC-PapersOnLine</em> 52 (26): 32–37. <a
href="https://doi.org/10.1016/j.ifacol.2019.12.232">https://doi.org/10.1016/j.ifacol.2019.12.232</a>.
</div>
<div id="ref-VillaverdeRai2019" class="csl-entry" role="listitem">
Villaverde, Alejandro F., Elba Raimúndez, Jan Hasenauer, and Julio R.
Banga. 2019. <span>“A Comparison of Methods for Quantifying Prediction
Uncertainty in Systems Biology.”</span> <em>IFAC-PapersOnLine</em> 52
(26): 45–51. <a
href="https://doi.org/10.1016/j.ifacol.2019.12.234">https://doi.org/10.1016/j.ifacol.2019.12.234</a>.
</div>
<div id="ref-WangSta2019" class="csl-entry" role="listitem">
Wang, Dantong, Paul Stapor, and Jan Hasenauer. 2019. <span>“Dirac
Mixture Distributions for the Approximation of Mixed Effects
Models.”</span> <em>IFAC-PapersOnLine</em> 52 (26): 200–206. <a
href="https://doi.org/10.1016/j.ifacol.2019.12.258">https://doi.org/10.1016/j.ifacol.2019.12.258</a>.
</div>
<div id="ref-Watanabe2019" class="csl-entry" role="listitem">
Watanabe, Simon Berglund. 2019. <span>“Identiﬁability of Parameters in
PBPK Models.”</span> Master’s thesis, Chalmers University of Technology
/ Department of Mathematical Sciences. <a
href="https://hdl.handle.net/20.500.12380/256855">https://hdl.handle.net/20.500.12380/256855</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2018</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-BallnusSch2018" class="csl-entry" role="listitem">
Ballnus, Benjamin, Steffen Schaper, Fabian J Theis, and Jan Hasenauer.
2018. <span>“Bayesian Parameter Estimation for Biochemical Reaction
Networks Using Region-Based Adaptive Parallel Tempering.”</span>
<em>Bioinformatics</em> 34 (13): i494–501. <a
href="https://doi.org/10.1093/bioinformatics/bty229">https://doi.org/10.1093/bioinformatics/bty229</a>.
</div>
<div id="ref-BastCal2018" class="csl-entry" role="listitem">
Bast, Lisa, Filippo Calzolari, Michael Strasser, Jan Hasenauer, Fabian
J. Theis, Jovica Ninkovic, and Carsten Marr. 2018. <span>“Subtle Changes
in Clonal Dynamics Underlie the Age-Related Decline in
Neurogenesis.”</span> <em>Cell Reports</em>. <a
href="https://doi.org/10.1016/j.celrep.2018.11.088">https://doi.org/10.1016/j.celrep.2018.11.088</a>.
</div>
<div id="ref-FroehlichKes2018" class="csl-entry" role="listitem">
Fröhlich, Fabian, Thomas Kessler, Daniel Weindl, Alexey Shadrin, Leonard
Schmiester, Hendrik Hache, Artur Muradyan, et al. 2018. <span>“Efficient
Parameter Estimation Enables the Prediction of Drug Response Using a
Mechanistic Pan-Cancer Pathway Model.”</span> <em>Cell Systems</em> 7
(6): 567–579.e6. <a
href="https://doi.org/10.1016/j.cels.2018.10.013">https://doi.org/10.1016/j.cels.2018.10.013</a>.
</div>
<div id="ref-FroehlichRei2018" class="csl-entry" role="listitem">
Fröhlich, Fabian, Anita Reiser, Laura Fink, Daniel Woschée, Thomas
Ligon, Fabian Joachim Theis, Joachim Oskar Rädler, and Jan Hasenauer.
2018. <span>“Multi-Experiment Nonlinear Mixed Effect Modeling of
Single-Cell Translation Kinetics After Transfection.”</span> <em>Npj
Systems Biology and Applications</em> 5 (1): 1. <a
href="https://doi.org/10.1038/s41540-018-0079-7">https://doi.org/10.1038/s41540-018-0079-7</a>.
</div>
<div id="ref-KaltenbacherPed2018" class="csl-entry" role="listitem">
Kaltenbacher, Barbara, and Barbara Pedretscher. 2018. <span>“Parameter
Estimation in SDEs via the Fokker–Planck Equation: Likelihood Function
and Adjoint Based Gradient Computation.”</span> <em>Journal of
Mathematical Analysis and Applications</em> 465 (2): 872–84. <a
href="https://doi.org/10.1016/j.jmaa.2018.05.048">https://doi.org/10.1016/j.jmaa.2018.05.048</a>.
</div>
<div id="ref-LoosKra2018" class="csl-entry" role="listitem">
Loos, Carolin, Sabrina Krause, and Jan Hasenauer. 2018.
<span>“Hierarchical Optimization for the Efficient Parametrization of
<span>ODE</span> Models.”</span> <em>Bioinformatics</em> 34 (24):
4266–73. <a
href="https://doi.org/10.1093/bioinformatics/bty514">https://doi.org/10.1093/bioinformatics/bty514</a>.
</div>
<div id="ref-LoosMoe2018" class="csl-entry" role="listitem">
Loos, Carolin, Katharina Moeller, Fabian Fröhlich, Tim Hucho, and Jan
Hasenauer. 2018. <span>“A Hierarchical, Data-Driven Approach to Modeling
Single-Cell Populations Predicts Latent Causes of Cell-to-Cell
Variability.”</span> <em>Cell Systems</em> 6 (5): 593–603. <a
href="https://doi.org/10.1016/j.cels.2018.04.008">https://doi.org/10.1016/j.cels.2018.04.008</a>.
</div>
<div id="ref-PittGom2018" class="csl-entry" role="listitem">
Pitt, Jake Alan, Lucian Gomoescu, Constantinos C. Pantelides, Benoît
Chachuat, and Julio R. Banga. 2018. <span>“Critical Assessment of
Parameter Estimation Methods in Models of Biological
Oscillators.”</span> <em>IFAC-PapersOnLine</em> 51 (19): 72–75. <a
href="https://doi.org/10.1016/j.ifacol.2018.09.040">https://doi.org/10.1016/j.ifacol.2018.09.040</a>.
</div>
<div id="ref-SchaelteSta2018" class="csl-entry" role="listitem">
Schälte, Y., P. Stapor, and J. Hasenauer. 2018. <span>“Evaluation of
Derivative-Free Optimizers for Parameter Estimation in Systems
Biology.”</span> <em>FAC-PapersOnLine</em> 51 (19): 98–101. <a
href="https://doi.org/10.1016/j.ifacol.2018.09.025">https://doi.org/10.1016/j.ifacol.2018.09.025</a>.
</div>
<div id="ref-StaporFro2018" class="csl-entry" role="listitem">
Stapor, Paul, Fabian Fröhlich, and Jan Hasenauer. 2018.
<span>“Optimization and Profile Calculation of <span>ODE</span> Models
Using Second Order Adjoint Sensitivity Analysis.”</span>
<em>Bioinformatics</em> 34 (13): i151–59. <a
href="https://doi.org/10.1093/bioinformatics/bty230">https://doi.org/10.1093/bioinformatics/bty230</a>.
</div>
<div id="ref-VillaverdeFroe2018" class="csl-entry" role="listitem">
Villaverde, Alejandro F, Fabian Fröhlich, Daniel Weindl, Jan Hasenauer,
and Julio R Banga. 2018. <span>“<span class="nocase">Benchmarking
optimization methods for parameter estimation in large kinetic
models</span>.”</span> <em>Bioinformatics</em> 35 (5): 830–38. <a
href="https://doi.org/10.1093/bioinformatics/bty736">https://doi.org/10.1093/bioinformatics/bty736</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2017</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-BallnusHug2017" class="csl-entry" role="listitem">
Ballnus, B., S. Hug, K. Hatz, L. Görlitz, J. Hasenauer, and F. J. Theis.
2017. <span>“Comprehensive Benchmarking of <span>Markov</span> Chain
<span>Monte</span> <span>Carlo</span> Methods for Dynamical
Systems.”</span> <em><span>BMC</span> Syst. Biol.</em> 11 (63). <a
href="https://doi.org/10.1186/s12918-017-0433-1">https://doi.org/10.1186/s12918-017-0433-1</a>.
</div>
<div id="ref-FroehlichKal2017" class="csl-entry" role="listitem">
Fröhlich, F., B. Kaltenbacher, F. J. Theis, and J. Hasenauer. 2017.
<span>“Scalable Parameter Estimation for Genome-Scale Biochemical
Reaction Networks.”</span> <em><span>PLoS</span> Comput. Biol.</em> 13
(1): e1005331. <a
href="https://doi.org/10.1371/journal.pcbi.1005331">https://doi.org/10.1371/journal.pcbi.1005331</a>.
</div>
<div id="ref-FroehlichThe2016" class="csl-entry" role="listitem">
Fröhlich, F., F. J. Theis, J. O. Rädler, and J. Hasenauer. 2017.
<span>“Parameter Estimation for Dynamical Systems with Discrete Events
and Logical Operations.”</span> <em>Bioinformatics</em> 33 (7): 1049–56.
<a
href="https://doi.org/10.1093/bioinformatics/btw764">https://doi.org/10.1093/bioinformatics/btw764</a>.
</div>
<div id="ref-KazeroonianThe2017" class="csl-entry" role="listitem">
Kazeroonian, A., F. J. Theis, and J. Hasenauer. 2017. <span>“A Scalable
Moment-Closure Approximation for Large-Scale Biochemical Reaction
Networks.”</span> <em>Bioinformatics</em> 33 (14): i293–300. <a
href="https://doi.org/10.1093/bioinformatics/btx249">https://doi.org/10.1093/bioinformatics/btx249</a>.
</div>
<div id="ref-MaierLoo2017" class="csl-entry" role="listitem">
Maier, C., C. Loos, and J. Hasenauer. 2017. <span>“Robust Parameter
Estimation for Dynamical Systems from Outlier-Corrupted Data.”</span>
<em>Bioinformatics</em> 33 (5): 718–25. <a
href="https://doi.org/10.1093/bioinformatics/btw703">https://doi.org/10.1093/bioinformatics/btw703</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2016</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-BoigerHas2016" class="csl-entry" role="listitem">
Boiger, R., J. Hasenauer, S. Hross, and B. Kaltenbacher. 2016.
<span>“Integration Based Profile Likelihood Calculation for
<span>PDE</span> Constrained Parameter Estimation Problems.”</span>
<em>Inverse Prob.</em> 32 (12): 125009. <a
href="https://doi.org/10.1088/0266-5611/32/12/125009">https://doi.org/10.1088/0266-5611/32/12/125009</a>.
</div>
<div id="ref-FiedlerRae2016" class="csl-entry" role="listitem">
Fiedler, A., S. Raeth, F. J. Theis, A. Hausser, and J. Hasenauer. 2016.
<span>“Tailored Parameter Optimization Methods for Ordinary Differential
Equation Models with Steady-State Constraints.”</span>
<em><span>BMC</span> Syst. Biol.</em> 10 (80). <a
href="https://doi.org/10.1186/s12918-016-0319-7">https://doi.org/10.1186/s12918-016-0319-7</a>.
</div>
<div id="ref-FroehlichTho2016" class="csl-entry" role="listitem">
Fröhlich, F., P. Thomas, A. Kazeroonian, F. J. Theis, R. Grima, and J.
Hasenauer. 2016. <span>“Inference for Stochastic Chemical Kinetics Using
Moment Equations and System Size Expansion.”</span>
<em><span>PLoS</span> Comput. Biol.</em> 12 (7): e1005030. <a
href="https://doi.org/10.1371/journal.pcbi.1005030">https://doi.org/10.1371/journal.pcbi.1005030</a>.
</div>
<div id="ref-HrossFie2016" class="csl-entry" role="listitem">
Hross, S., A. Fiedler, F. J. Theis, and J. Hasenauer. 2016.
<span>“Quantitative Comparison of Competing <span>PDE</span> Models for
<span>Pom1p</span> Dynamics in Fission Yeast.”</span> In <em>Proc. 6th
<span>IFAC</span> Conf. Found. Syst. Biol. Eng.</em>, edited by R.
Findeisen, E. Bullinger, E. Balsa-Canto, and K. Bernaerts, 49:264–69.
26. IFAC-PapersOnLine. <a
href="https://doi.org/10.1016/j.ifacol.2016.12.136">https://doi.org/10.1016/j.ifacol.2016.12.136</a>.
</div>
<div id="ref-KazeroonianFro2016" class="csl-entry" role="listitem">
Kazeroonian, A., F. Fröhlich, A. Raue, F. J. Theis, and J. Hasenauer.
2016. <span>“<span>CERENA:</span> <span>Chemical</span>
<span>REaction</span> Network <span>Analyzer – A</span> Toolbox for the
Simulation and Analysis of Stochastic Chemical Kinetics.”</span>
<em><span>PLoS</span> <span>ONE</span></em> 11 (1): e0146732. <a
href="https://doi.org/10.1371/journal.pone.0146732">https://doi.org/10.1371/journal.pone.0146732</a>.
</div>
<div id="ref-LoosFie2016" class="csl-entry" role="listitem">
Loos, C., A. Fiedler, and J. Hasenauer. 2016. <span>“Parameter
Estimation for Reaction Rate Equation Constrained Mixture
Models.”</span> In <em>Proc. 13th Int. Conf. Comp. Meth. Syst.
Biol.</em>, edited by E. Bartocci, P. Lio, and N. Paoletti, 186–200.
Lecture Notes in Bioinformatics. Springer International Publishing. <a
href="https://doi.org/10.1007/978-3-319-45177-0">https://doi.org/10.1007/978-3-319-45177-0</a>.
</div>
</div>
<h1 class="unnumbered" id="section">2015</h1>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-LoosMar2015" class="csl-entry" role="listitem">
Loos, C., C. Marr, F. J. Theis, and J. Hasenauer. 2015.
<span>“Computational Methods in Systems Biology.”</span> In, edited by
O. Roux and J. Bourdon, 9308:52–63. Lecture Notes in Computer Science.
Springer International Publishing. <a
href="https://doi.org/10.1007/978-3-319-23401-4_6">https://doi.org/10.1007/978-3-319-23401-4_6</a>.
</div>
</div>
